  INFO root [0x00001520] [2018-07-02 15:14:29,897] - 
  INFO root [0x00001520] [2018-07-02 15:14:29,946] - -----------------------------------------------------------------------------------------


  INFO root [0x00001520] [2018-07-02 15:14:31,567] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00001520] [2018-07-02 15:14:31,568] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00001520] [2018-07-02 15:14:31,568] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00001520] [2018-07-02 15:14:32,175] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x00001520] [2018-07-02 15:14:32,176] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x00001520] [2018-07-02 15:14:32,179] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:10998 )
  INFO root [0x00001520] [2018-07-02 15:14:32,179] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:10998, SOMAXCONN=128.
  INFO root [0x00001520] [2018-07-02 15:14:32,179] - server-configs:
  INFO root [0x00001520] [2018-07-02 15:14:32,179] - 	gameUpdateHertz : 10
  INFO root [0x00001520] [2018-07-02 15:14:32,179] - 	entryScriptFile : kbemain
  INFO root [0x00001520] [2018-07-02 15:14:32,179] - 	internalAddr : 192.168.31.56:10998
  INFO root [0x00001520] [2018-07-02 15:14:32,180] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x00001520] [2018-07-02 15:14:32,252] - 	componentID : 7001
  INFO root [0x00001520] [2018-07-02 15:14:33,449] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 15410. PID: 7500 ----
  INFO root [0x00001520] [2018-07-02 15:14:33,450] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00001520] [2018-07-02 15:14:33,579] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001520] [2018-07-02 15:14:33,626] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001520] [2018-07-02 15:14:39,476] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001520] [2018-07-02 15:14:39,630] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001520] [2018-07-02 15:14:39,776] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001520] [2018-07-02 15:14:41,245] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001520] [2018-07-02 15:14:41,546] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001520] [2018-07-02 15:14:41,641] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001520] [2018-07-02 15:14:41,769] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001520] [2018-07-02 15:14:42,406] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001520] [2018-07-02 15:14:43,075] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001520] [2018-07-02 15:14:43,076] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001520] [2018-07-02 15:14:43,846] - loaded script:Account(1).
 DEBUG root [0x00001520] [2018-07-02 15:14:43,986] - loaded script:Hall(2).
 DEBUG root [0x00001520] [2018-07-02 15:14:44,026] - loaded script:BattleField(3).
 DEBUG root [0x00001520] [2018-07-02 15:14:44,059] - loaded script:Avatar(4).
  INFO root [0x00001520] [2018-07-02 15:14:44,061] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00001520] [2018-07-02 15:14:44,127] - TelnetServer server is running on port 40000
  INFO root [0x00001520] [2018-07-02 15:14:45,005] - ---- baseapp is running ----
 DEBUG root [0x00001520] [2018-07-02 15:14:45,149] - Components::process(): Request for the process of identity...
  WARN root [0x00001520] [2018-07-02 15:14:51,510] - EntityApp::checkTickPeriod: tick took 6495% (6.49 seconds)!
  INFO root [0x00001520] [2018-07-02 15:14:52,568] - Components::findComponents: find logger(1)...
  INFO root [0x00001520] [2018-07-02 15:14:52,631] - Components::findComponents: found logger, addr:192.168.31.56:10989
  INFO root [0x00001520] [2018-07-02 15:14:52,632] - Components::addComponent: reset orderLog, uid=15410!
  INFO root [0x00001520] [2018-07-02 15:14:52,632] - Components::addComponent[logger], uid=15410, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001520] [2018-07-02 15:14:52,634] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00001520] [2018-07-02 15:14:52,736] - Components::findComponents: found dbmgr, addr:192.168.31.56:10993
  INFO root [0x00001520] [2018-07-02 15:14:52,736] - Components::addComponent: reset orderLog, uid=15410!
  INFO root [0x00001520] [2018-07-02 15:14:52,736] - Components::addComponent[dbmgr], uid=15410, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001520] [2018-07-02 15:14:52,737] - Components::findComponents: found dbmgr, addr:192.168.31.56:10993
  WARN root [0x00001520] [2018-07-02 15:14:53,737] - Forwarding logs to logger[192.168.31.56:10989/0/0/0]...
  INFO root [0x00002034] [2018-07-02 16:30:53,957] - 
  INFO root [0x00002034] [2018-07-02 16:30:53,958] - -----------------------------------------------------------------------------------------


  INFO root [0x00002034] [2018-07-02 16:30:53,960] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00002034] [2018-07-02 16:30:53,961] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00002034] [2018-07-02 16:30:53,961] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00002034] [2018-07-02 16:30:53,984] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20016 )
  INFO root [0x00002034] [2018-07-02 16:30:53,986] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20016, SOMAXCONN=128.
  INFO root [0x00002034] [2018-07-02 16:30:53,990] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:12248 )
  INFO root [0x00002034] [2018-07-02 16:30:53,991] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:12248, SOMAXCONN=128.
  INFO root [0x00002034] [2018-07-02 16:30:53,991] - server-configs:
  INFO root [0x00002034] [2018-07-02 16:30:53,991] - 	gameUpdateHertz : 10
  INFO root [0x00002034] [2018-07-02 16:30:54,006] - 	entryScriptFile : kbemain
  INFO root [0x00002034] [2018-07-02 16:30:54,007] - 	internalAddr : 192.168.31.56:12248
  INFO root [0x00002034] [2018-07-02 16:30:54,008] - 	externalAddr : 192.168.31.56:20016
  INFO root [0x00002034] [2018-07-02 16:30:54,008] - 	componentID : 7001
  INFO root [0x00002034] [2018-07-02 16:30:56,069] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 30435. PID: 5472 ----
  INFO root [0x00002034] [2018-07-02 16:30:56,073] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00002034] [2018-07-02 16:30:56,101] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002034] [2018-07-02 16:30:56,102] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00002034] [2018-07-02 16:30:56,278] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002034] [2018-07-02 16:30:56,281] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00002034] [2018-07-02 16:30:56,282] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00002034] [2018-07-02 16:30:56,289] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002034] [2018-07-02 16:30:56,290] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00002034] [2018-07-02 16:30:56,290] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002034] [2018-07-02 16:30:56,291] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002034] [2018-07-02 16:30:56,508] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00002034] [2018-07-02 16:30:56,694] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00002034] [2018-07-02 16:30:56,694] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00002034] [2018-07-02 16:30:56,880] - loaded script:Account(1).
 DEBUG root [0x00002034] [2018-07-02 16:30:56,884] - loaded script:Hall(2).
 DEBUG root [0x00002034] [2018-07-02 16:30:56,886] - loaded script:BattleField(3).
 DEBUG root [0x00002034] [2018-07-02 16:30:56,891] - loaded script:Avatar(4).
  INFO root [0x00002034] [2018-07-02 16:30:56,894] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00002034] [2018-07-02 16:30:56,898] - TelnetServer server is running on port 40001
  INFO root [0x00002034] [2018-07-02 16:30:58,225] - ---- baseapp is running ----
 DEBUG root [0x00002034] [2018-07-02 16:30:58,226] - Components::process(): Request for the process of identity...
 ERROR root [0x00002034] [2018-07-02 16:30:58,367] - Components::process: found baseapp, addr:192.168.31.56:10998
 ERROR root [0x00002034] [2018-07-02 16:30:58,368] - ServerApp::onIdentityillegal: The current process and baseapp(componentID=7001 ->conflicted???, pid=7500, addr=192.168.31.56) conflict, the process will exit!
Can modify the components-CID and UID to avoid conflict.
  INFO root [0x00002034] [2018-07-02 16:30:58,369] - Shutdowner::onShutdownBegin: shutting down(period=1, tickPeriod=1)
S_INFO root [0x00002034] [2018-07-02 16:30:58,370] - onBaseAppShutDown: state=0
  INFO root [0x00002034] [2018-07-02 16:30:59,375] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00002034] [2018-07-02 16:30:59,375] - onBaseAppShutDown: state=1
S_INFO root [0x00002034] [2018-07-02 16:31:00,376] - onReadyForShutDown()
  INFO root [0x00002034] [2018-07-02 16:31:00,376] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00002034] [2018-07-02 16:31:00,377] - onBaseAppShutDown: state=2
 DEBUG root [0x00002034] [2018-07-02 16:31:00,389] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002034] [2018-07-02 16:31:00,390] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00002034] [2018-07-02 16:31:00,390] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00002034] [2018-07-02 16:31:00,390] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002034] [2018-07-02 16:31:00,390] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00002034] [2018-07-02 16:31:00,390] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002034] [2018-07-02 16:31:00,391] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002034] [2018-07-02 16:31:00,391] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00002034] [2018-07-02 16:31:00,402] - Script::uninstall(): is successfully!
 DEBUG root [0x00002034] [2018-07-02 16:31:00,402] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00002034] [2018-07-02 16:31:00,703] - ThreadPool::destroy(): waiting for thread(2)[0x472268,0x472200,], try=1
 DEBUG root [0x00002034] [2018-07-02 16:31:01,104] - TPThread::~TPThread(): 0x472268
 DEBUG root [0x00002034] [2018-07-02 16:31:01,104] - TPThread::~TPThread(): 0x472200
 DEBUG root [0x00002034] [2018-07-02 16:31:01,104] - ThreadPool::destroy(): successfully!
 DEBUG root [0x00002034] [2018-07-02 16:31:01,105] - Bundle::destroyObjPool(): size 5.
 DEBUG root [0x00002034] [2018-07-02 16:31:01,105] - Channel::destroyObjPool(): size 0.
 DEBUG root [0x00002034] [2018-07-02 16:31:01,105] - TCPPacket::destroyObjPool(): size 0.
 DEBUG root [0x00002034] [2018-07-02 16:31:01,105] - UDPPacket::destroyObjPool(): size 12.
 DEBUG root [0x00002034] [2018-07-02 16:31:01,106] - EndPoint::destroyObjPool(): size 0.
 DEBUG root [0x00002034] [2018-07-02 16:31:01,106] - Address::destroyObjPool(): size 0.
 DEBUG root [0x00002034] [2018-07-02 16:31:01,106] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [0x00002034] [2018-07-02 16:31:01,106] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [0x00002034] [2018-07-02 16:31:01,107] - baseapp(7001) has shut down.
  INFO root [0x00002034] [2018-07-02 16:31:02,120] - EventDispatcher()::~EventDispatcher: Num timers = 2
  INFO root [0x00002708] [2018-07-02 16:31:07,358] - 
  INFO root [0x00002708] [2018-07-02 16:31:07,359] - -----------------------------------------------------------------------------------------


  INFO root [0x00002708] [2018-07-02 16:31:07,361] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00002708] [2018-07-02 16:31:07,361] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00002708] [2018-07-02 16:31:07,362] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00002708] [2018-07-02 16:31:07,398] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20016 )
  INFO root [0x00002708] [2018-07-02 16:31:07,398] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20016, SOMAXCONN=128.
  INFO root [0x00002708] [2018-07-02 16:31:07,429] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:12269 )
  INFO root [0x00002708] [2018-07-02 16:31:07,430] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:12269, SOMAXCONN=128.
  INFO root [0x00002708] [2018-07-02 16:31:07,430] - server-configs:
  INFO root [0x00002708] [2018-07-02 16:31:07,430] - 	gameUpdateHertz : 10
  INFO root [0x00002708] [2018-07-02 16:31:07,431] - 	entryScriptFile : kbemain
  INFO root [0x00002708] [2018-07-02 16:31:07,431] - 	internalAddr : 192.168.31.56:12269
  INFO root [0x00002708] [2018-07-02 16:31:07,431] - 	externalAddr : 192.168.31.56:20016
  INFO root [0x00002708] [2018-07-02 16:31:07,431] - 	componentID : 7001
  INFO root [0x00002708] [2018-07-02 16:31:09,397] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 30481. PID: 8204 ----
  INFO root [0x00002708] [2018-07-02 16:31:09,399] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00002708] [2018-07-02 16:31:09,399] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002708] [2018-07-02 16:31:09,400] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00002708] [2018-07-02 16:31:09,609] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002708] [2018-07-02 16:31:09,610] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00002708] [2018-07-02 16:31:09,611] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00002708] [2018-07-02 16:31:09,619] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002708] [2018-07-02 16:31:09,620] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00002708] [2018-07-02 16:31:09,620] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002708] [2018-07-02 16:31:09,622] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002708] [2018-07-02 16:31:09,747] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00002708] [2018-07-02 16:31:09,862] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00002708] [2018-07-02 16:31:09,863] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00002708] [2018-07-02 16:31:10,088] - loaded script:Account(1).
 DEBUG root [0x00002708] [2018-07-02 16:31:10,092] - loaded script:Hall(2).
 DEBUG root [0x00002708] [2018-07-02 16:31:10,095] - loaded script:BattleField(3).
 DEBUG root [0x00002708] [2018-07-02 16:31:10,137] - loaded script:Avatar(4).
  INFO root [0x00002708] [2018-07-02 16:31:10,141] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00002708] [2018-07-02 16:31:10,143] - TelnetServer server is running on port 40001
  INFO root [0x00002708] [2018-07-02 16:31:11,917] - ---- baseapp is running ----
 DEBUG root [0x00002708] [2018-07-02 16:31:11,919] - Components::process(): Request for the process of identity...
 ERROR root [0x00002708] [2018-07-02 16:31:12,061] - Components::process: found baseapp, addr:192.168.31.56:10998
 ERROR root [0x00002708] [2018-07-02 16:31:12,063] - ServerApp::onIdentityillegal: The current process and baseapp(componentID=7001 ->conflicted???, pid=7500, addr=192.168.31.56) conflict, the process will exit!
Can modify the components-CID and UID to avoid conflict.
  INFO root [0x00002708] [2018-07-02 16:31:12,064] - Shutdowner::onShutdownBegin: shutting down(period=1, tickPeriod=1)
S_INFO root [0x00002708] [2018-07-02 16:31:12,065] - onBaseAppShutDown: state=0
  INFO root [0x00002708] [2018-07-02 16:31:13,065] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00002708] [2018-07-02 16:31:13,066] - onBaseAppShutDown: state=1
S_INFO root [0x00002708] [2018-07-02 16:31:14,067] - onReadyForShutDown()
  INFO root [0x00002708] [2018-07-02 16:31:14,067] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00002708] [2018-07-02 16:31:14,068] - onBaseAppShutDown: state=2
 DEBUG root [0x00002708] [2018-07-02 16:31:14,080] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002708] [2018-07-02 16:31:14,081] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00002708] [2018-07-02 16:31:14,081] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00002708] [2018-07-02 16:31:14,081] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002708] [2018-07-02 16:31:14,081] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00002708] [2018-07-02 16:31:14,081] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002708] [2018-07-02 16:31:14,081] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002708] [2018-07-02 16:31:14,082] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00002708] [2018-07-02 16:31:14,094] - Script::uninstall(): is successfully!
 DEBUG root [0x00002708] [2018-07-02 16:31:14,095] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00002708] [2018-07-02 16:31:14,395] - ThreadPool::destroy(): waiting for thread(2)[0x482258,0x4821f0,], try=1
 DEBUG root [0x00002708] [2018-07-02 16:31:14,795] - TPThread::~TPThread(): 0x482258
 DEBUG root [0x00002708] [2018-07-02 16:31:14,795] - TPThread::~TPThread(): 0x4821f0
 DEBUG root [0x00002708] [2018-07-02 16:31:14,795] - ThreadPool::destroy(): successfully!
 DEBUG root [0x00002708] [2018-07-02 16:31:14,795] - Bundle::destroyObjPool(): size 5.
 DEBUG root [0x00002708] [2018-07-02 16:31:14,796] - Channel::destroyObjPool(): size 0.
 DEBUG root [0x00002708] [2018-07-02 16:31:14,796] - TCPPacket::destroyObjPool(): size 0.
 DEBUG root [0x00002708] [2018-07-02 16:31:14,796] - UDPPacket::destroyObjPool(): size 12.
 DEBUG root [0x00002708] [2018-07-02 16:31:14,796] - EndPoint::destroyObjPool(): size 0.
 DEBUG root [0x00002708] [2018-07-02 16:31:14,796] - Address::destroyObjPool(): size 0.
 DEBUG root [0x00002708] [2018-07-02 16:31:14,797] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [0x00002708] [2018-07-02 16:31:14,797] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [0x00002708] [2018-07-02 16:31:14,797] - baseapp(7001) has shut down.
  INFO root [0x00002708] [2018-07-02 16:31:15,798] - EventDispatcher()::~EventDispatcher: Num timers = 2
  INFO root [0x000018d0] [2018-07-02 16:32:44,022] - 
  INFO root [0x000018d0] [2018-07-02 16:32:44,028] - -----------------------------------------------------------------------------------------


  INFO root [0x000018d0] [2018-07-02 16:32:44,029] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x000018d0] [2018-07-02 16:32:44,029] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x000018d0] [2018-07-02 16:32:44,029] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x000018d0] [2018-07-02 16:32:44,105] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20016 )
  INFO root [0x000018d0] [2018-07-02 16:32:44,105] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20016, SOMAXCONN=128.
  INFO root [0x000018d0] [2018-07-02 16:32:44,120] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:12316 )
  INFO root [0x000018d0] [2018-07-02 16:32:44,120] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:12316, SOMAXCONN=128.
  INFO root [0x000018d0] [2018-07-02 16:32:44,120] - server-configs:
  INFO root [0x000018d0] [2018-07-02 16:32:44,121] - 	gameUpdateHertz : 10
  INFO root [0x000018d0] [2018-07-02 16:32:44,121] - 	entryScriptFile : kbemain
  INFO root [0x000018d0] [2018-07-02 16:32:44,121] - 	internalAddr : 192.168.31.56:12316
  INFO root [0x000018d0] [2018-07-02 16:32:44,121] - 	externalAddr : 192.168.31.56:20016
  INFO root [0x000018d0] [2018-07-02 16:32:44,122] - 	componentID : 7001
  INFO root [0x000018d0] [2018-07-02 16:32:46,147] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 30794. PID: 10340 ----
  INFO root [0x000018d0] [2018-07-02 16:32:46,149] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x000018d0] [2018-07-02 16:32:46,149] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x000018d0] [2018-07-02 16:32:46,150] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x000018d0] [2018-07-02 16:32:46,376] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000018d0] [2018-07-02 16:32:46,377] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x000018d0] [2018-07-02 16:32:46,378] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x000018d0] [2018-07-02 16:32:46,386] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x000018d0] [2018-07-02 16:32:46,387] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x000018d0] [2018-07-02 16:32:46,387] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x000018d0] [2018-07-02 16:32:46,389] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x000018d0] [2018-07-02 16:32:46,572] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x000018d0] [2018-07-02 16:32:46,726] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x000018d0] [2018-07-02 16:32:46,727] - Script::install(): KBExtra is successfully!
 DEBUG root [0x000018d0] [2018-07-02 16:32:46,966] - loaded script:Account(1).
 DEBUG root [0x000018d0] [2018-07-02 16:32:46,970] - loaded script:Hall(2).
 DEBUG root [0x000018d0] [2018-07-02 16:32:46,973] - loaded script:BattleField(3).
 DEBUG root [0x000018d0] [2018-07-02 16:32:46,976] - loaded script:Avatar(4).
  INFO root [0x000018d0] [2018-07-02 16:32:46,980] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x000018d0] [2018-07-02 16:32:46,981] - TelnetServer server is running on port 40001
  INFO root [0x000018d0] [2018-07-02 16:32:47,780] - ---- baseapp is running ----
 DEBUG root [0x000018d0] [2018-07-02 16:32:47,781] - Components::process(): Request for the process of identity...
 ERROR root [0x000018d0] [2018-07-02 16:32:47,983] - Components::process: found baseapp, addr:192.168.31.56:10998
 ERROR root [0x000018d0] [2018-07-02 16:32:47,986] - ServerApp::onIdentityillegal: The current process and baseapp(componentID=7001 ->conflicted???, pid=7500, addr=192.168.31.56) conflict, the process will exit!
Can modify the components-CID and UID to avoid conflict.
  INFO root [0x000018d0] [2018-07-02 16:32:47,988] - Shutdowner::onShutdownBegin: shutting down(period=1, tickPeriod=1)
S_INFO root [0x000018d0] [2018-07-02 16:32:47,989] - onBaseAppShutDown: state=0
  WARN root [0x000018d0] [2018-07-02 16:32:47,994] - EntityApp::checkTickPeriod: tick took 213% (0.21 seconds)!
  INFO root [0x000018d0] [2018-07-02 16:32:48,989] - Shutdowner::onShutdown: shutting down
S_INFO root [0x000018d0] [2018-07-02 16:32:48,990] - onBaseAppShutDown: state=1
S_INFO root [0x000018d0] [2018-07-02 16:32:49,991] - onReadyForShutDown()
  INFO root [0x000018d0] [2018-07-02 16:32:49,992] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x000018d0] [2018-07-02 16:32:49,992] - onBaseAppShutDown: state=2
 DEBUG root [0x000018d0] [2018-07-02 16:32:50,004] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000018d0] [2018-07-02 16:32:50,005] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x000018d0] [2018-07-02 16:32:50,005] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x000018d0] [2018-07-02 16:32:50,005] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x000018d0] [2018-07-02 16:32:50,005] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x000018d0] [2018-07-02 16:32:50,006] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x000018d0] [2018-07-02 16:32:50,006] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x000018d0] [2018-07-02 16:32:50,006] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x000018d0] [2018-07-02 16:32:50,017] - Script::uninstall(): is successfully!
 DEBUG root [0x000018d0] [2018-07-02 16:32:50,018] - ThreadPool::destroy(): starting size 2.
  WARN root [0x000018d0] [2018-07-02 16:32:50,318] - ThreadPool::destroy(): waiting for thread(2)[0x5b2258,0x5b21f0,], try=1
 DEBUG root [0x000018d0] [2018-07-02 16:32:50,718] - TPThread::~TPThread(): 0x5b2258
 DEBUG root [0x000018d0] [2018-07-02 16:32:50,718] - TPThread::~TPThread(): 0x5b21f0
 DEBUG root [0x000018d0] [2018-07-02 16:32:50,718] - ThreadPool::destroy(): successfully!
 DEBUG root [0x000018d0] [2018-07-02 16:32:50,718] - Bundle::destroyObjPool(): size 4.
 DEBUG root [0x000018d0] [2018-07-02 16:32:50,719] - Channel::destroyObjPool(): size 0.
 DEBUG root [0x000018d0] [2018-07-02 16:32:50,719] - TCPPacket::destroyObjPool(): size 0.
 DEBUG root [0x000018d0] [2018-07-02 16:32:50,719] - UDPPacket::destroyObjPool(): size 12.
 DEBUG root [0x000018d0] [2018-07-02 16:32:50,719] - EndPoint::destroyObjPool(): size 0.
 DEBUG root [0x000018d0] [2018-07-02 16:32:50,720] - Address::destroyObjPool(): size 0.
 DEBUG root [0x000018d0] [2018-07-02 16:32:50,720] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [0x000018d0] [2018-07-02 16:32:50,720] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [0x000018d0] [2018-07-02 16:32:50,720] - baseapp(7001) has shut down.
  INFO root [0x000018d0] [2018-07-02 16:32:51,721] - EventDispatcher()::~EventDispatcher: Num timers = 2
  INFO root [0x0000126c] [2018-07-02 18:03:14,738] - 
  INFO root [0x0000126c] [2018-07-02 18:03:14,753] - -----------------------------------------------------------------------------------------


  INFO root [0x0000126c] [2018-07-02 18:03:14,753] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x0000126c] [2018-07-02 18:03:14,753] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x0000126c] [2018-07-02 18:03:14,753] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x0000126c] [2018-07-02 18:03:14,784] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x0000126c] [2018-07-02 18:03:14,784] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x0000126c] [2018-07-02 18:03:14,784] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:1342 )
  INFO root [0x0000126c] [2018-07-02 18:03:14,800] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:1342, SOMAXCONN=128.
  INFO root [0x0000126c] [2018-07-02 18:03:14,800] - server-configs:
  INFO root [0x0000126c] [2018-07-02 18:03:14,800] - 	gameUpdateHertz : 10
  INFO root [0x0000126c] [2018-07-02 18:03:14,800] - 	entryScriptFile : kbemain
  INFO root [0x0000126c] [2018-07-02 18:03:14,800] - 	internalAddr : 192.168.31.56:1342
  INFO root [0x0000126c] [2018-07-02 18:03:14,800] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x0000126c] [2018-07-02 18:03:14,800] - 	componentID : 7001
  INFO root [0x0000126c] [2018-07-02 18:03:16,844] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 15762. PID: 4712 ----
  INFO root [0x0000126c] [2018-07-02 18:03:16,844] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x0000126c] [2018-07-02 18:03:16,844] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x0000126c] [2018-07-02 18:03:16,844] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x0000126c] [2018-07-02 18:03:17,015] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x0000126c] [2018-07-02 18:03:17,015] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x0000126c] [2018-07-02 18:03:17,015] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x0000126c] [2018-07-02 18:03:17,031] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x0000126c] [2018-07-02 18:03:17,031] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x0000126c] [2018-07-02 18:03:17,031] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x0000126c] [2018-07-02 18:03:17,031] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x0000126c] [2018-07-02 18:03:17,187] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x0000126c] [2018-07-02 18:03:17,296] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x0000126c] [2018-07-02 18:03:17,296] - Script::install(): KBExtra is successfully!
 DEBUG root [0x0000126c] [2018-07-02 18:03:17,421] - loaded script:Account(1).
 DEBUG root [0x0000126c] [2018-07-02 18:03:17,421] - loaded script:Hall(2).
 DEBUG root [0x0000126c] [2018-07-02 18:03:17,436] - loaded script:BattleField(3).
 DEBUG root [0x0000126c] [2018-07-02 18:03:17,436] - loaded script:Avatar(4).
  INFO root [0x0000126c] [2018-07-02 18:03:17,436] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x0000126c] [2018-07-02 18:03:17,436] - TelnetServer server is running on port 40000
  INFO root [0x0000126c] [2018-07-02 18:03:18,419] - ---- baseapp is running ----
 DEBUG root [0x0000126c] [2018-07-02 18:03:18,419] - Components::process(): Request for the process of identity...
  WARN root [0x0000126c] [2018-07-02 18:03:24,659] - EntityApp::checkTickPeriod: tick took 6241% (6.24 seconds)!
  INFO root [0x0000126c] [2018-07-02 18:03:25,673] - Components::findComponents: find logger(1)...
  INFO root [0x0000126c] [2018-07-02 18:03:25,736] - Components::findComponents: found logger, addr:192.168.31.56:1345
  INFO root [0x0000126c] [2018-07-02 18:03:25,736] - Components::addComponent: reset orderLog, uid=15762!
  INFO root [0x0000126c] [2018-07-02 18:03:25,736] - Components::addComponent[logger], uid=15762, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x0000126c] [2018-07-02 18:03:25,736] - Components::findComponents: find dbmgr(2)...
  INFO root [0x0000126c] [2018-07-02 18:03:25,845] - Components::findComponents: found dbmgr, addr:192.168.31.56:1343
  INFO root [0x0000126c] [2018-07-02 18:03:25,845] - Components::addComponent: reset orderLog, uid=15762!
  INFO root [0x0000126c] [2018-07-02 18:03:25,845] - Components::addComponent[dbmgr], uid=15762, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x0000126c] [2018-07-02 18:03:25,845] - Components::findComponents: found dbmgr, addr:192.168.31.56:1343
  WARN root [0x0000126c] [2018-07-02 18:03:26,859] - Forwarding logs to logger[192.168.31.56:1345/0/0/0]...
 ERROR root [0x0000126c] [2018-07-02 18:27:30,884] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x0000126c] [2018-07-02 18:27:30,885] - The following logs sent to logger failed:
  WARN root [0x0000126c] [2018-07-02 18:27:30,886] - ==> [2018-07-02 18:27:30 882] TCPPacketReceiver::processPendingEvents(192.168.31.56:1345): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x0000126c] [2018-07-02 18:27:30,887] - ==> [2018-07-02 18:27:30 885] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x0000126c] [2018-07-02 18:27:30,984] - TCPPacketReceiver::processPendingEvents(192.168.31.56:1343): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x0000126c] [2018-07-02 18:27:30,988] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x0000126c] [2018-07-02 18:27:30,994] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x0000126c] [2018-07-02 18:27:30,997] - onBaseAppShutDown: state=0
  INFO root [0x0000126c] [2018-07-02 18:27:30,998] - Shutdowner::onShutdown: shutting down
S_INFO root [0x0000126c] [2018-07-02 18:27:30,998] - onBaseAppShutDown: state=1
  INFO root [0x0000126c] [2018-07-02 18:27:30,999] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x0000126c] [2018-07-02 18:27:30,999] - onBaseAppShutDown: state=2
 ERROR root [0x0000126c] [2018-07-02 18:27:31,037] - Account::onCellWriteToDBCompleted(2002): not found dbmgr!
 ERROR root [0x0000126c] [2018-07-02 18:27:31,038] - Base::onDestroy: not found dbmgr!
 ERROR root [0x0000126c] [2018-07-02 18:27:31,040] - Account::onCellWriteToDBCompleted(2003): not found dbmgr!
 ERROR root [0x0000126c] [2018-07-02 18:27:31,042] - Base::onDestroy: not found dbmgr!
 DEBUG root [0x0000126c] [2018-07-02 18:27:31,044] - BattleField::~BattleField(): 2004
 DEBUG root [0x0000126c] [2018-07-02 18:27:31,044] - Hall::~Hall(): 2001
 ERROR root [0x0000126c] [2018-07-02 18:27:31,045] - EntityGarbages::~EntityGarbages(): leaked, size=4.
 ERROR root [0x0000126c] [2018-07-02 18:27:31,046] - 	--> leaked: Account(2002).
 ERROR root [0x0000126c] [2018-07-02 18:27:31,055] - 	--> leaked: Account(2003).
 ERROR root [0x0000126c] [2018-07-02 18:27:31,056] - 	--> leaked: Avatar(2005).
 ERROR root [0x0000126c] [2018-07-02 18:27:31,057] - 	--> leaked: Avatar(2006).
 ERROR root [0x0000126c] [2018-07-02 18:27:31,061] - PyGC::debugTracing(): FixedArray : leaked(14)
 ERROR root [0x0000126c] [2018-07-02 18:27:31,062] - PyGC::debugTracing(): Base : leaked(4)
 ERROR root [0x0000126c] [2018-07-02 18:27:31,064] - PyGC::debugTracing(): FixedDict : leaked(8)
 ERROR root [0x0000126c] [2018-07-02 18:27:31,066] - PyGC::debugTracing(): EntityMailbox : leaked(9)
 DEBUG root [0x0000126c] [2018-07-02 18:27:31,068] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x0000126c] [2018-07-02 18:27:31,068] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x0000126c] [2018-07-02 18:27:31,068] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x0000126c] [2018-07-02 18:27:31,069] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x0000126c] [2018-07-02 18:27:31,069] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x0000126c] [2018-07-02 18:27:31,069] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x0000126c] [2018-07-02 18:27:31,070] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x0000126c] [2018-07-02 18:27:31,070] - Script::uninstall_py_dlls(): PyInit_unicodedata
 DEBUG root [0x0000126c] [2018-07-02 18:27:31,076] - Avatar::~Avatar(): 2006
 DEBUG root [0x0000126c] [2018-07-02 18:27:31,076] - Account::~Account(): 2002
 DEBUG root [0x0000126c] [2018-07-02 18:27:31,077] - Avatar::~Avatar(): 2005
 DEBUG root [0x0000126c] [2018-07-02 18:27:31,078] - Account::~Account(): 2003
  INFO root [0x0000126c] [2018-07-02 18:27:31,097] - Script::uninstall(): is successfully!
 DEBUG root [0x0000126c] [2018-07-02 18:27:31,097] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00000cc8] [2018-07-02 18:27:36,967] - 
  INFO root [0x00000cc8] [2018-07-02 18:27:36,968] - -----------------------------------------------------------------------------------------


  INFO root [0x00000cc8] [2018-07-02 18:27:36,970] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00000cc8] [2018-07-02 18:27:36,970] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00000cc8] [2018-07-02 18:27:36,971] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00000cc8] [2018-07-02 18:27:36,995] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x00000cc8] [2018-07-02 18:27:36,995] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x00000cc8] [2018-07-02 18:27:37,010] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:1972 )
  INFO root [0x00000cc8] [2018-07-02 18:27:37,011] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:1972, SOMAXCONN=128.
  INFO root [0x00000cc8] [2018-07-02 18:27:37,012] - server-configs:
  INFO root [0x00000cc8] [2018-07-02 18:27:37,012] - 	gameUpdateHertz : 10
  INFO root [0x00000cc8] [2018-07-02 18:27:37,012] - 	entryScriptFile : kbemain
  INFO root [0x00000cc8] [2018-07-02 18:27:37,013] - 	internalAddr : 192.168.31.56:1972
  INFO root [0x00000cc8] [2018-07-02 18:27:37,013] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x00000cc8] [2018-07-02 18:27:37,013] - 	componentID : 7001
  INFO root [0x00000cc8] [2018-07-02 18:27:37,871] - Components::findLogger: found logger, addr:192.168.31.56:1968
  INFO root [0x00000cc8] [2018-07-02 18:27:37,871] - Components::addComponent: reset orderLog, uid=20536!
  INFO root [0x00000cc8] [2018-07-02 18:27:37,872] - Components::addComponent[logger], uid=20536, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00000cc8] [2018-07-02 18:27:37,886] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 20536. PID: 9144 ----
  INFO root [0x00000cc8] [2018-07-02 18:27:37,887] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00000cc8] [2018-07-02 18:27:37,887] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00000cc8] [2018-07-02 18:27:37,888] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00000cc8] [2018-07-02 18:27:38,061] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00000cc8] [2018-07-02 18:27:38,062] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00000cc8] [2018-07-02 18:27:38,064] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00000cc8] [2018-07-02 18:27:38,104] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00000cc8] [2018-07-02 18:27:38,105] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00000cc8] [2018-07-02 18:27:38,106] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00000cc8] [2018-07-02 18:27:38,107] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00000cc8] [2018-07-02 18:27:38,195] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00000cc8] [2018-07-02 18:27:38,290] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00000cc8] [2018-07-02 18:27:38,291] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00000cc8] [2018-07-02 18:27:38,427] - loaded script:Account(1).
 DEBUG root [0x00000cc8] [2018-07-02 18:27:38,430] - loaded script:Hall(2).
 DEBUG root [0x00000cc8] [2018-07-02 18:27:38,433] - loaded script:BattleField(3).
 DEBUG root [0x00000cc8] [2018-07-02 18:27:38,435] - loaded script:Avatar(4).
  INFO root [0x00000cc8] [2018-07-02 18:27:38,438] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00000cc8] [2018-07-02 18:27:38,440] - TelnetServer server is running on port 40000
  INFO root [0x00000cc8] [2018-07-02 18:27:40,086] - ---- baseapp is running ----
 DEBUG root [0x00000cc8] [2018-07-02 18:27:40,087] - Components::process(): Request for the process of identity...
  WARN root [0x00000cc8] [2018-07-02 18:27:46,211] - Forwarding logs to logger[192.168.31.56:1968/0/0/0]...
 ERROR root [0x00000cc8] [2018-07-02 18:39:17,152] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00000cc8] [2018-07-02 18:39:17,152] - The following logs sent to logger failed:
  WARN root [0x00000cc8] [2018-07-02 18:39:17,152] - ==> [2018-07-02 18:39:17 142] TCPPacketReceiver::processPendingEvents(192.168.31.56:1968): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00000cc8] [2018-07-02 18:39:17,152] - ==> [2018-07-02 18:39:17 152] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00000cc8] [2018-07-02 18:39:17,262] - TCPPacketReceiver::processPendingEvents(192.168.31.56:1969): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00000cc8] [2018-07-02 18:39:17,272] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00000cc8] [2018-07-02 18:39:17,282] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00000cc8] [2018-07-02 18:39:17,282] - onBaseAppShutDown: state=0
  INFO root [0x00000cc8] [2018-07-02 18:39:17,292] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00000cc8] [2018-07-02 18:39:17,292] - onBaseAppShutDown: state=1
  INFO root [0x00000cc8] [2018-07-02 18:39:17,292] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00000cc8] [2018-07-02 18:39:17,292] - onBaseAppShutDown: state=2
 ERROR root [0x00000cc8] [2018-07-02 18:39:17,322] - Account::onCellWriteToDBCompleted(2): not found dbmgr!
 ERROR root [0x00000cc8] [2018-07-02 18:39:17,322] - Base::onDestroy: not found dbmgr!
 ERROR root [0x00000cc8] [2018-07-02 18:39:17,322] - Account::onCellWriteToDBCompleted(3): not found dbmgr!
 ERROR root [0x00000cc8] [2018-07-02 18:39:17,322] - Base::onDestroy: not found dbmgr!
 DEBUG root [0x00000cc8] [2018-07-02 18:39:17,322] - BattleField::~BattleField(): 4
 DEBUG root [0x00000cc8] [2018-07-02 18:39:17,322] - Avatar::~Avatar(): 5
 DEBUG root [0x00000cc8] [2018-07-02 18:39:17,322] - Avatar::~Avatar(): 6
 DEBUG root [0x00000cc8] [2018-07-02 18:39:17,322] - BattleField::~BattleField(): 7
 DEBUG root [0x00000cc8] [2018-07-02 18:39:17,332] - Hall::~Hall(): 1
 ERROR root [0x00000cc8] [2018-07-02 18:39:17,332] - EntityGarbages::~EntityGarbages(): leaked, size=4.
 ERROR root [0x00000cc8] [2018-07-02 18:39:17,332] - 	--> leaked: Avatar(9).
 ERROR root [0x00000cc8] [2018-07-02 18:39:17,332] - 	--> leaked: Account(2).
 ERROR root [0x00000cc8] [2018-07-02 18:39:17,332] - 	--> leaked: Account(3).
 ERROR root [0x00000cc8] [2018-07-02 18:39:17,332] - 	--> leaked: Avatar(8).
 ERROR root [0x00000cc8] [2018-07-02 18:39:17,332] - PyGC::debugTracing(): FixedArray : leaked(12)
 ERROR root [0x00000cc8] [2018-07-02 18:39:17,332] - PyGC::debugTracing(): Base : leaked(4)
 ERROR root [0x00000cc8] [2018-07-02 18:39:17,342] - PyGC::debugTracing(): FixedDict : leaked(8)
 ERROR root [0x00000cc8] [2018-07-02 18:39:17,342] - PyGC::debugTracing(): EntityMailbox : leaked(3)
 DEBUG root [0x00000cc8] [2018-07-02 18:39:17,342] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00000cc8] [2018-07-02 18:39:17,342] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00000cc8] [2018-07-02 18:39:17,342] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00000cc8] [2018-07-02 18:39:17,342] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00000cc8] [2018-07-02 18:39:17,342] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00000cc8] [2018-07-02 18:39:17,342] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00000cc8] [2018-07-02 18:39:17,342] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00000cc8] [2018-07-02 18:39:17,342] - Script::uninstall_py_dlls(): PyInit_unicodedata
 DEBUG root [0x00000cc8] [2018-07-02 18:39:17,342] - Avatar::~Avatar(): 8
 DEBUG root [0x00000cc8] [2018-07-02 18:39:17,342] - Account::~Account(): 2
 DEBUG root [0x00000cc8] [2018-07-02 18:39:17,342] - Avatar::~Avatar(): 9
 DEBUG root [0x00000cc8] [2018-07-02 18:39:17,342] - Account::~Account(): 3
  INFO root [0x00000cc8] [2018-07-02 18:39:17,362] - Script::uninstall(): is successfully!
 DEBUG root [0x00000cc8] [2018-07-02 18:39:17,362] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00001530] [2018-07-02 19:11:16,855] - 
  INFO root [0x00001530] [2018-07-02 19:11:16,860] - -----------------------------------------------------------------------------------------


  INFO root [0x00001530] [2018-07-02 19:11:16,862] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00001530] [2018-07-02 19:11:16,863] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00001530] [2018-07-02 19:11:16,864] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00001530] [2018-07-02 19:11:16,888] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x00001530] [2018-07-02 19:11:16,889] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x00001530] [2018-07-02 19:11:16,893] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:2524 )
  INFO root [0x00001530] [2018-07-02 19:11:16,894] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:2524, SOMAXCONN=128.
  INFO root [0x00001530] [2018-07-02 19:11:16,894] - server-configs:
  INFO root [0x00001530] [2018-07-02 19:11:16,894] - 	gameUpdateHertz : 10
  INFO root [0x00001530] [2018-07-02 19:11:16,895] - 	entryScriptFile : kbemain
  INFO root [0x00001530] [2018-07-02 19:11:16,895] - 	internalAddr : 192.168.31.56:2524
  INFO root [0x00001530] [2018-07-02 19:11:16,895] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x00001530] [2018-07-02 19:11:16,896] - 	componentID : 7001
  INFO root [0x00001530] [2018-07-02 19:11:18,893] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 29092. PID: 5416 ----
  INFO root [0x00001530] [2018-07-02 19:11:18,894] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00001530] [2018-07-02 19:11:18,897] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001530] [2018-07-02 19:11:18,898] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001530] [2018-07-02 19:11:19,115] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001530] [2018-07-02 19:11:19,117] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001530] [2018-07-02 19:11:19,118] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001530] [2018-07-02 19:11:19,126] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001530] [2018-07-02 19:11:19,127] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001530] [2018-07-02 19:11:19,127] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001530] [2018-07-02 19:11:19,129] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001530] [2018-07-02 19:11:19,305] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001530] [2018-07-02 19:11:19,513] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001530] [2018-07-02 19:11:19,513] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001530] [2018-07-02 19:11:19,783] - loaded script:Account(1).
 DEBUG root [0x00001530] [2018-07-02 19:11:19,787] - loaded script:Hall(2).
 DEBUG root [0x00001530] [2018-07-02 19:11:19,791] - loaded script:BattleField(3).
 DEBUG root [0x00001530] [2018-07-02 19:11:19,796] - loaded script:Avatar(4).
  INFO root [0x00001530] [2018-07-02 19:11:19,799] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00001530] [2018-07-02 19:11:19,801] - TelnetServer server is running on port 40000
  INFO root [0x00001530] [2018-07-02 19:11:20,802] - ---- baseapp is running ----
 DEBUG root [0x00001530] [2018-07-02 19:11:20,803] - Components::process(): Request for the process of identity...
  WARN root [0x00001530] [2018-07-02 19:11:26,900] - EntityApp::checkTickPeriod: tick took 6097% (6.10 seconds)!
  INFO root [0x00001530] [2018-07-02 19:11:27,977] - Components::findComponents: find logger(1)...
  INFO root [0x00001530] [2018-07-02 19:11:28,010] - Components::findComponents: found logger, addr:192.168.31.56:2525
  INFO root [0x00001530] [2018-07-02 19:11:28,011] - Components::addComponent: reset orderLog, uid=29092!
  INFO root [0x00001530] [2018-07-02 19:11:28,011] - Components::addComponent[logger], uid=29092, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001530] [2018-07-02 19:11:28,013] - Components::findComponents: find dbmgr(2)...
  WARN root [0x00001530] [2018-07-02 19:11:29,115] - Forwarding logs to logger[192.168.31.56:2525/0/0/0]...
 ERROR root [0x00001530] [2018-07-02 19:11:31,739] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00001530] [2018-07-02 19:11:31,744] - The following logs sent to logger failed:
  WARN root [0x00001530] [2018-07-02 19:11:31,745] - ==> [2018-07-02 19:11:31 737] TCPPacketReceiver::processPendingEvents(192.168.31.56:2525): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001530] [2018-07-02 19:11:31,745] - ==> [2018-07-02 19:11:31 740] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00001fdc] [2018-07-02 19:11:53,585] - 
  INFO root [0x00001fdc] [2018-07-02 19:11:53,587] - -----------------------------------------------------------------------------------------


  INFO root [0x00001fdc] [2018-07-02 19:11:53,589] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00001fdc] [2018-07-02 19:11:53,589] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00001fdc] [2018-07-02 19:11:53,590] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00001fdc] [2018-07-02 19:11:53,605] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x00001fdc] [2018-07-02 19:11:53,606] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x00001fdc] [2018-07-02 19:11:53,610] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:2559 )
  INFO root [0x00001fdc] [2018-07-02 19:11:53,610] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:2559, SOMAXCONN=128.
  INFO root [0x00001fdc] [2018-07-02 19:11:53,611] - server-configs:
  INFO root [0x00001fdc] [2018-07-02 19:11:53,611] - 	gameUpdateHertz : 10
  INFO root [0x00001fdc] [2018-07-02 19:11:53,611] - 	entryScriptFile : kbemain
  INFO root [0x00001fdc] [2018-07-02 19:11:53,611] - 	internalAddr : 192.168.31.56:2559
  INFO root [0x00001fdc] [2018-07-02 19:11:53,612] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x00001fdc] [2018-07-02 19:11:53,612] - 	componentID : 7001
  INFO root [0x00001fdc] [2018-07-02 19:11:55,660] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 29213. PID: 9972 ----
  INFO root [0x00001fdc] [2018-07-02 19:11:55,661] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00001fdc] [2018-07-02 19:11:55,662] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001fdc] [2018-07-02 19:11:55,663] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001fdc] [2018-07-02 19:11:55,878] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001fdc] [2018-07-02 19:11:55,879] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001fdc] [2018-07-02 19:11:55,880] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001fdc] [2018-07-02 19:11:55,888] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001fdc] [2018-07-02 19:11:55,889] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001fdc] [2018-07-02 19:11:55,889] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001fdc] [2018-07-02 19:11:55,891] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001fdc] [2018-07-02 19:11:56,075] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001fdc] [2018-07-02 19:11:56,195] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001fdc] [2018-07-02 19:11:56,195] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001fdc] [2018-07-02 19:11:56,333] - loaded script:Account(1).
 DEBUG root [0x00001fdc] [2018-07-02 19:11:56,337] - loaded script:Hall(2).
 DEBUG root [0x00001fdc] [2018-07-02 19:11:56,341] - loaded script:BattleField(3).
 DEBUG root [0x00001fdc] [2018-07-02 19:11:56,345] - loaded script:Avatar(4).
  INFO root [0x00001fdc] [2018-07-02 19:11:56,348] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00001fdc] [2018-07-02 19:11:56,351] - TelnetServer server is running on port 40000
  INFO root [0x00001fdc] [2018-07-02 19:11:57,531] - ---- baseapp is running ----
 DEBUG root [0x00001fdc] [2018-07-02 19:11:57,531] - Components::process(): Request for the process of identity...
  WARN root [0x00001fdc] [2018-07-02 19:12:03,629] - EntityApp::checkTickPeriod: tick took 6098% (6.10 seconds)!
  INFO root [0x00001fdc] [2018-07-02 19:12:04,692] - Components::findComponents: find logger(1)...
  INFO root [0x00001fdc] [2018-07-02 19:12:04,778] - Components::findComponents: found logger, addr:192.168.31.56:2562
  INFO root [0x00001fdc] [2018-07-02 19:12:04,778] - Components::addComponent: reset orderLog, uid=29213!
  INFO root [0x00001fdc] [2018-07-02 19:12:04,779] - Components::addComponent[logger], uid=29213, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001fdc] [2018-07-02 19:12:04,781] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00001fdc] [2018-07-02 19:12:04,886] - Components::findComponents: found dbmgr, addr:192.168.31.56:2554
  INFO root [0x00001fdc] [2018-07-02 19:12:04,886] - Components::addComponent: reset orderLog, uid=29213!
  INFO root [0x00001fdc] [2018-07-02 19:12:04,886] - Components::addComponent[dbmgr], uid=29213, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001fdc] [2018-07-02 19:12:04,887] - Components::findComponents: found dbmgr, addr:192.168.31.56:2554
  WARN root [0x00001fdc] [2018-07-02 19:12:05,887] - Forwarding logs to logger[192.168.31.56:2562/0/0/0]...
 ERROR root [0x00001fdc] [2018-07-02 19:15:51,908] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00001fdc] [2018-07-02 19:15:51,916] - The following logs sent to logger failed:
  WARN root [0x00001fdc] [2018-07-02 19:15:51,917] - ==> [2018-07-02 19:15:51 908] TCPPacketReceiver::processPendingEvents(192.168.31.56:2562): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001fdc] [2018-07-02 19:15:51,917] - ==> [2018-07-02 19:15:51 909] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00001fdc] [2018-07-02 19:15:52,022] - TCPPacketReceiver::processPendingEvents(192.168.31.56:2554): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001fdc] [2018-07-02 19:15:52,027] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00001fdc] [2018-07-02 19:15:52,029] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00001fdc] [2018-07-02 19:15:52,030] - onBaseAppShutDown: state=0
  INFO root [0x00001fdc] [2018-07-02 19:15:52,031] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00001fdc] [2018-07-02 19:15:52,032] - onBaseAppShutDown: state=1
  INFO root [0x00001fdc] [2018-07-02 19:15:52,032] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00001fdc] [2018-07-02 19:15:52,033] - onBaseAppShutDown: state=2
 ERROR root [0x00001fdc] [2018-07-02 19:15:52,069] - Account::onCellWriteToDBCompleted(2002): not found dbmgr!
 ERROR root [0x00001fdc] [2018-07-02 19:15:52,070] - Base::onDestroy: not found dbmgr!
 ERROR root [0x00001fdc] [2018-07-02 19:15:52,072] - Account::onCellWriteToDBCompleted(2003): not found dbmgr!
 ERROR root [0x00001fdc] [2018-07-02 19:15:52,074] - Base::onDestroy: not found dbmgr!
 DEBUG root [0x00001fdc] [2018-07-02 19:15:52,075] - BattleField::~BattleField(): 2004
 DEBUG root [0x00001fdc] [2018-07-02 19:15:52,076] - Hall::~Hall(): 2001
 ERROR root [0x00001fdc] [2018-07-02 19:15:52,076] - EntityGarbages::~EntityGarbages(): leaked, size=4.
 ERROR root [0x00001fdc] [2018-07-02 19:15:52,077] - 	--> leaked: Account(2002).
 ERROR root [0x00001fdc] [2018-07-02 19:15:52,079] - 	--> leaked: Account(2003).
 ERROR root [0x00001fdc] [2018-07-02 19:15:52,080] - 	--> leaked: Avatar(2005).
 ERROR root [0x00001fdc] [2018-07-02 19:15:52,081] - 	--> leaked: Avatar(2006).
 ERROR root [0x00001fdc] [2018-07-02 19:15:52,086] - PyGC::debugTracing(): FixedArray : leaked(13)
 ERROR root [0x00001fdc] [2018-07-02 19:15:52,087] - PyGC::debugTracing(): Base : leaked(4)
 ERROR root [0x00001fdc] [2018-07-02 19:15:52,089] - PyGC::debugTracing(): FixedDict : leaked(8)
 ERROR root [0x00001fdc] [2018-07-02 19:15:52,091] - PyGC::debugTracing(): EntityMailbox : leaked(5)
 DEBUG root [0x00001fdc] [2018-07-02 19:15:52,094] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001fdc] [2018-07-02 19:15:52,095] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00001fdc] [2018-07-02 19:15:52,096] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00001fdc] [2018-07-02 19:15:52,096] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001fdc] [2018-07-02 19:15:52,097] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00001fdc] [2018-07-02 19:15:52,097] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001fdc] [2018-07-02 19:15:52,097] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001fdc] [2018-07-02 19:15:52,098] - Script::uninstall_py_dlls(): PyInit_unicodedata
 DEBUG root [0x00001fdc] [2018-07-02 19:15:52,103] - Avatar::~Avatar(): 2006
 DEBUG root [0x00001fdc] [2018-07-02 19:15:52,104] - Account::~Account(): 2002
 DEBUG root [0x00001fdc] [2018-07-02 19:15:52,104] - Avatar::~Avatar(): 2005
 DEBUG root [0x00001fdc] [2018-07-02 19:15:52,105] - Account::~Account(): 2003
  INFO root [0x00001fdc] [2018-07-02 19:15:52,123] - Script::uninstall(): is successfully!
 DEBUG root [0x00001fdc] [2018-07-02 19:15:52,124] - ThreadPool::destroy(): starting size 2.
  INFO root [0x000020c0] [2018-07-02 19:15:57,123] - 
  INFO root [0x000020c0] [2018-07-02 19:15:57,126] - -----------------------------------------------------------------------------------------


  INFO root [0x000020c0] [2018-07-02 19:15:57,128] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x000020c0] [2018-07-02 19:15:57,128] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x000020c0] [2018-07-02 19:15:57,129] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x000020c0] [2018-07-02 19:15:57,190] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x000020c0] [2018-07-02 19:15:57,191] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x000020c0] [2018-07-02 19:15:57,195] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:2705 )
  INFO root [0x000020c0] [2018-07-02 19:15:57,196] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:2705, SOMAXCONN=128.
  INFO root [0x000020c0] [2018-07-02 19:15:57,196] - server-configs:
  INFO root [0x000020c0] [2018-07-02 19:15:57,196] - 	gameUpdateHertz : 10
  INFO root [0x000020c0] [2018-07-02 19:15:57,196] - 	entryScriptFile : kbemain
  INFO root [0x000020c0] [2018-07-02 19:15:57,197] - 	internalAddr : 192.168.31.56:2705
  INFO root [0x000020c0] [2018-07-02 19:15:57,197] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x000020c0] [2018-07-02 19:15:57,197] - 	componentID : 7001
  INFO root [0x000020c0] [2018-07-02 19:15:58,409] - Components::findLogger: found logger, addr:192.168.31.56:2704
  INFO root [0x000020c0] [2018-07-02 19:15:58,410] - Components::addComponent: reset orderLog, uid=30009!
  INFO root [0x000020c0] [2018-07-02 19:15:58,411] - Components::addComponent[logger], uid=30009, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000020c0] [2018-07-02 19:15:58,424] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 30009. PID: 5396 ----
  INFO root [0x000020c0] [2018-07-02 19:15:58,425] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x000020c0] [2018-07-02 19:15:58,426] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x000020c0] [2018-07-02 19:15:58,426] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x000020c0] [2018-07-02 19:15:58,596] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000020c0] [2018-07-02 19:15:58,597] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x000020c0] [2018-07-02 19:15:58,598] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x000020c0] [2018-07-02 19:15:58,605] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x000020c0] [2018-07-02 19:15:58,606] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x000020c0] [2018-07-02 19:15:58,606] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x000020c0] [2018-07-02 19:15:58,608] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x000020c0] [2018-07-02 19:15:58,727] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x000020c0] [2018-07-02 19:15:58,818] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x000020c0] [2018-07-02 19:15:58,818] - Script::install(): KBExtra is successfully!
 DEBUG root [0x000020c0] [2018-07-02 19:15:58,959] - loaded script:Account(1).
 DEBUG root [0x000020c0] [2018-07-02 19:15:58,963] - loaded script:Hall(2).
 DEBUG root [0x000020c0] [2018-07-02 19:15:58,966] - loaded script:BattleField(3).
 DEBUG root [0x000020c0] [2018-07-02 19:15:58,969] - loaded script:Avatar(4).
  INFO root [0x000020c0] [2018-07-02 19:15:58,973] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x000020c0] [2018-07-02 19:15:58,974] - TelnetServer server is running on port 40000
  INFO root [0x000020c0] [2018-07-02 19:16:00,430] - ---- baseapp is running ----
 DEBUG root [0x000020c0] [2018-07-02 19:16:00,447] - Components::process(): Request for the process of identity...
  WARN root [0x000020c0] [2018-07-02 19:16:06,575] - Forwarding logs to logger[192.168.31.56:2704/0/0/0]...
 ERROR root [0x000020c0] [2018-07-02 19:20:02,132] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x000020c0] [2018-07-02 19:20:02,143] - The following logs sent to logger failed:
  WARN root [0x000020c0] [2018-07-02 19:20:02,143] - ==> [2018-07-02 19:20:02 131] TCPPacketReceiver::processPendingEvents(192.168.31.56:2704): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000020c0] [2018-07-02 19:20:02,144] - ==> [2018-07-02 19:20:02 133] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x000020c0] [2018-07-02 19:20:02,236] - TCPPacketReceiver::processPendingEvents(192.168.31.56:2708): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000020c0] [2018-07-02 19:20:02,240] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x000020c0] [2018-07-02 19:20:02,243] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x000020c0] [2018-07-02 19:20:02,246] - onBaseAppShutDown: state=0
  INFO root [0x000020c0] [2018-07-02 19:20:02,246] - Shutdowner::onShutdown: shutting down
S_INFO root [0x000020c0] [2018-07-02 19:20:02,247] - onBaseAppShutDown: state=1
  INFO root [0x000020c0] [2018-07-02 19:20:02,247] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x000020c0] [2018-07-02 19:20:02,247] - onBaseAppShutDown: state=2
 ERROR root [0x000020c0] [2018-07-02 19:20:02,283] - Account::onCellWriteToDBCompleted(2): not found dbmgr!
 ERROR root [0x000020c0] [2018-07-02 19:20:02,286] - Base::onDestroy: not found dbmgr!
 ERROR root [0x000020c0] [2018-07-02 19:20:02,289] - Account::onCellWriteToDBCompleted(3): not found dbmgr!
 ERROR root [0x000020c0] [2018-07-02 19:20:02,290] - Base::onDestroy: not found dbmgr!
 DEBUG root [0x000020c0] [2018-07-02 19:20:02,292] - BattleField::~BattleField(): 4
 DEBUG root [0x000020c0] [2018-07-02 19:20:02,293] - Hall::~Hall(): 1
 ERROR root [0x000020c0] [2018-07-02 19:20:02,293] - EntityGarbages::~EntityGarbages(): leaked, size=4.
 ERROR root [0x000020c0] [2018-07-02 19:20:02,295] - 	--> leaked: Account(2).
 ERROR root [0x000020c0] [2018-07-02 19:20:02,296] - 	--> leaked: Account(3).
 ERROR root [0x000020c0] [2018-07-02 19:20:02,297] - 	--> leaked: Avatar(5).
 ERROR root [0x000020c0] [2018-07-02 19:20:02,298] - 	--> leaked: Avatar(6).
 ERROR root [0x000020c0] [2018-07-02 19:20:02,302] - PyGC::debugTracing(): FixedArray : leaked(14)
 ERROR root [0x000020c0] [2018-07-02 19:20:02,303] - PyGC::debugTracing(): Base : leaked(4)
 ERROR root [0x000020c0] [2018-07-02 19:20:02,306] - PyGC::debugTracing(): FixedDict : leaked(8)
 ERROR root [0x000020c0] [2018-07-02 19:20:02,307] - PyGC::debugTracing(): EntityMailbox : leaked(7)
 DEBUG root [0x000020c0] [2018-07-02 19:20:02,309] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000020c0] [2018-07-02 19:20:02,309] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x000020c0] [2018-07-02 19:20:02,310] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x000020c0] [2018-07-02 19:20:02,310] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x000020c0] [2018-07-02 19:20:02,310] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x000020c0] [2018-07-02 19:20:02,310] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x000020c0] [2018-07-02 19:20:02,311] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x000020c0] [2018-07-02 19:20:02,311] - Script::uninstall_py_dlls(): PyInit_unicodedata
 DEBUG root [0x000020c0] [2018-07-02 19:20:02,317] - Avatar::~Avatar(): 5
 DEBUG root [0x000020c0] [2018-07-02 19:20:02,318] - Account::~Account(): 2
 DEBUG root [0x000020c0] [2018-07-02 19:20:02,318] - Avatar::~Avatar(): 6
 DEBUG root [0x000020c0] [2018-07-02 19:20:02,319] - Account::~Account(): 3
  INFO root [0x000020c0] [2018-07-02 19:20:02,340] - Script::uninstall(): is successfully!
 DEBUG root [0x000020c0] [2018-07-02 19:20:02,340] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00001fdc] [2018-07-02 19:21:08,856] - 
  INFO root [0x00001fdc] [2018-07-02 19:21:08,860] - -----------------------------------------------------------------------------------------


  INFO root [0x00001fdc] [2018-07-02 19:21:08,863] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00001fdc] [2018-07-02 19:21:08,863] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00001fdc] [2018-07-02 19:21:08,864] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00001fdc] [2018-07-02 19:21:08,876] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x00001fdc] [2018-07-02 19:21:08,877] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x00001fdc] [2018-07-02 19:21:08,881] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:2839 )
  INFO root [0x00001fdc] [2018-07-02 19:21:08,882] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:2839, SOMAXCONN=128.
  INFO root [0x00001fdc] [2018-07-02 19:21:08,882] - server-configs:
  INFO root [0x00001fdc] [2018-07-02 19:21:08,882] - 	gameUpdateHertz : 10
  INFO root [0x00001fdc] [2018-07-02 19:21:08,883] - 	entryScriptFile : kbemain
  INFO root [0x00001fdc] [2018-07-02 19:21:08,883] - 	internalAddr : 192.168.31.56:2839
  INFO root [0x00001fdc] [2018-07-02 19:21:08,883] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x00001fdc] [2018-07-02 19:21:08,883] - 	componentID : 7001
  INFO root [0x00001fdc] [2018-07-02 19:21:10,035] - Components::findLogger: found logger, addr:192.168.31.56:2843
  INFO root [0x00001fdc] [2018-07-02 19:21:10,036] - Components::addComponent: reset orderLog, uid=31028!
  INFO root [0x00001fdc] [2018-07-02 19:21:10,037] - Components::addComponent[logger], uid=31028, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001fdc] [2018-07-02 19:21:10,045] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 31028. PID: 4568 ----
  INFO root [0x00001fdc] [2018-07-02 19:21:10,046] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00001fdc] [2018-07-02 19:21:10,047] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001fdc] [2018-07-02 19:21:10,047] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001fdc] [2018-07-02 19:21:10,203] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001fdc] [2018-07-02 19:21:10,204] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001fdc] [2018-07-02 19:21:10,204] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001fdc] [2018-07-02 19:21:10,212] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001fdc] [2018-07-02 19:21:10,212] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001fdc] [2018-07-02 19:21:10,212] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001fdc] [2018-07-02 19:21:10,213] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001fdc] [2018-07-02 19:21:10,284] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001fdc] [2018-07-02 19:21:10,360] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001fdc] [2018-07-02 19:21:10,360] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001fdc] [2018-07-02 19:21:10,455] - loaded script:Account(1).
 DEBUG root [0x00001fdc] [2018-07-02 19:21:10,458] - loaded script:Hall(2).
 DEBUG root [0x00001fdc] [2018-07-02 19:21:10,461] - loaded script:BattleField(3).
 DEBUG root [0x00001fdc] [2018-07-02 19:21:10,464] - loaded script:Avatar(4).
  INFO root [0x00001fdc] [2018-07-02 19:21:10,467] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00001fdc] [2018-07-02 19:21:10,469] - TelnetServer server is running on port 40000
  INFO root [0x00001fdc] [2018-07-02 19:21:12,584] - ---- baseapp is running ----
 DEBUG root [0x00001fdc] [2018-07-02 19:21:12,586] - Components::process(): Request for the process of identity...
  WARN root [0x00001fdc] [2018-07-02 19:21:18,715] - Forwarding logs to logger[192.168.31.56:2843/0/0/0]...
 ERROR root [0x00001fdc] [2018-07-02 20:16:29,605] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00001fdc] [2018-07-02 20:16:29,632] - The following logs sent to logger failed:
  WARN root [0x00001fdc] [2018-07-02 20:16:29,632] - ==> [2018-07-02 20:16:29 600] TCPPacketReceiver::processPendingEvents(192.168.31.56:2843): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001fdc] [2018-07-02 20:16:29,633] - ==> [2018-07-02 20:16:29 606] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00001fdc] [2018-07-02 20:16:29,732] - TCPPacketReceiver::processPendingEvents(192.168.31.56:2842): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001fdc] [2018-07-02 20:16:29,736] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00001fdc] [2018-07-02 20:16:29,741] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00001fdc] [2018-07-02 20:16:29,743] - onBaseAppShutDown: state=0
  INFO root [0x00001fdc] [2018-07-02 20:16:29,744] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00001fdc] [2018-07-02 20:16:29,744] - onBaseAppShutDown: state=1
  INFO root [0x00001fdc] [2018-07-02 20:16:29,744] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00001fdc] [2018-07-02 20:16:29,745] - onBaseAppShutDown: state=2
 ERROR root [0x00001fdc] [2018-07-02 20:16:29,778] - Account::onCellWriteToDBCompleted(2): not found dbmgr!
 ERROR root [0x00001fdc] [2018-07-02 20:16:29,779] - Base::onDestroy: not found dbmgr!
 ERROR root [0x00001fdc] [2018-07-02 20:16:29,780] - Account::onCellWriteToDBCompleted(3): not found dbmgr!
 ERROR root [0x00001fdc] [2018-07-02 20:16:29,781] - Base::onDestroy: not found dbmgr!
 DEBUG root [0x00001fdc] [2018-07-02 20:16:29,781] - BattleField::~BattleField(): 4
 DEBUG root [0x00001fdc] [2018-07-02 20:16:29,782] - Hall::~Hall(): 1
 ERROR root [0x00001fdc] [2018-07-02 20:16:29,782] - EntityGarbages::~EntityGarbages(): leaked, size=4.
 ERROR root [0x00001fdc] [2018-07-02 20:16:29,783] - 	--> leaked: Account(2).
 ERROR root [0x00001fdc] [2018-07-02 20:16:29,784] - 	--> leaked: Account(3).
 ERROR root [0x00001fdc] [2018-07-02 20:16:29,785] - 	--> leaked: Avatar(5).
 ERROR root [0x00001fdc] [2018-07-02 20:16:29,787] - 	--> leaked: Avatar(6).
 ERROR root [0x00001fdc] [2018-07-02 20:16:29,789] - PyGC::debugTracing(): FixedArray : leaked(14)
 ERROR root [0x00001fdc] [2018-07-02 20:16:29,790] - PyGC::debugTracing(): Base : leaked(4)
 ERROR root [0x00001fdc] [2018-07-02 20:16:29,791] - PyGC::debugTracing(): FixedDict : leaked(8)
 ERROR root [0x00001fdc] [2018-07-02 20:16:29,793] - PyGC::debugTracing(): EntityMailbox : leaked(5)
 DEBUG root [0x00001fdc] [2018-07-02 20:16:29,794] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001fdc] [2018-07-02 20:16:29,795] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00001fdc] [2018-07-02 20:16:29,795] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00001fdc] [2018-07-02 20:16:29,796] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001fdc] [2018-07-02 20:16:29,796] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00001fdc] [2018-07-02 20:16:29,796] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001fdc] [2018-07-02 20:16:29,797] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001fdc] [2018-07-02 20:16:29,797] - Script::uninstall_py_dlls(): PyInit_unicodedata
 DEBUG root [0x00001fdc] [2018-07-02 20:16:29,802] - Avatar::~Avatar(): 6
 DEBUG root [0x00001fdc] [2018-07-02 20:16:29,803] - Account::~Account(): 2
 DEBUG root [0x00001fdc] [2018-07-02 20:16:29,803] - Avatar::~Avatar(): 5
 DEBUG root [0x00001fdc] [2018-07-02 20:16:29,804] - Account::~Account(): 3
  INFO root [0x00001fdc] [2018-07-02 20:16:29,820] - Script::uninstall(): is successfully!
 DEBUG root [0x00001fdc] [2018-07-02 20:16:29,820] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00001d58] [2018-07-02 21:20:33,971] - 
  INFO root [0x00001d58] [2018-07-02 21:20:33,976] - -----------------------------------------------------------------------------------------


  INFO root [0x00001d58] [2018-07-02 21:20:33,979] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00001d58] [2018-07-02 21:20:33,979] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00001d58] [2018-07-02 21:20:33,980] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00001d58] [2018-07-02 21:20:34,108] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x00001d58] [2018-07-02 21:20:34,109] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x00001d58] [2018-07-02 21:20:34,163] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:3854 )
  INFO root [0x00001d58] [2018-07-02 21:20:34,164] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:3854, SOMAXCONN=128.
  INFO root [0x00001d58] [2018-07-02 21:20:34,164] - server-configs:
  INFO root [0x00001d58] [2018-07-02 21:20:34,164] - 	gameUpdateHertz : 10
  INFO root [0x00001d58] [2018-07-02 21:20:34,165] - 	entryScriptFile : kbemain
  INFO root [0x00001d58] [2018-07-02 21:20:34,165] - 	internalAddr : 192.168.31.56:3854
  INFO root [0x00001d58] [2018-07-02 21:20:34,165] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x00001d58] [2018-07-02 21:20:34,166] - 	componentID : 7001
  INFO root [0x00001d58] [2018-07-02 21:20:35,120] - Components::findLogger: found logger, addr:192.168.31.56:3851
  INFO root [0x00001d58] [2018-07-02 21:20:35,121] - Components::addComponent: reset orderLog, uid=21655!
  INFO root [0x00001d58] [2018-07-02 21:20:35,122] - Components::addComponent[logger], uid=21655, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001d58] [2018-07-02 21:20:35,133] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 21655. PID: 5928 ----
  INFO root [0x00001d58] [2018-07-02 21:20:35,133] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00001d58] [2018-07-02 21:20:35,135] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001d58] [2018-07-02 21:20:35,135] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001d58] [2018-07-02 21:20:35,305] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001d58] [2018-07-02 21:20:35,306] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001d58] [2018-07-02 21:20:35,307] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001d58] [2018-07-02 21:20:35,314] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001d58] [2018-07-02 21:20:35,315] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001d58] [2018-07-02 21:20:35,315] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001d58] [2018-07-02 21:20:35,317] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001d58] [2018-07-02 21:20:35,408] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001d58] [2018-07-02 21:20:35,513] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001d58] [2018-07-02 21:20:35,513] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001d58] [2018-07-02 21:20:35,686] - loaded script:Account(1).
 DEBUG root [0x00001d58] [2018-07-02 21:20:35,689] - loaded script:Hall(2).
 DEBUG root [0x00001d58] [2018-07-02 21:20:35,693] - loaded script:BattleField(3).
 DEBUG root [0x00001d58] [2018-07-02 21:20:35,696] - loaded script:Avatar(4).
  INFO root [0x00001d58] [2018-07-02 21:20:35,699] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00001d58] [2018-07-02 21:20:35,701] - TelnetServer server is running on port 40000
  INFO root [0x00001d58] [2018-07-02 21:20:37,569] - ---- baseapp is running ----
 DEBUG root [0x00001d58] [2018-07-02 21:20:37,570] - Components::process(): Request for the process of identity...
  INFO root [0x00001f74] [2018-07-02 21:20:42,662] - 
  INFO root [0x00001f74] [2018-07-02 21:20:42,664] - -----------------------------------------------------------------------------------------


  INFO root [0x00001f74] [2018-07-02 21:20:42,666] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00001f74] [2018-07-02 21:20:42,667] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00001f74] [2018-07-02 21:20:42,668] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00001f74] [2018-07-02 21:20:42,712] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x00001f74] [2018-07-02 21:20:42,713] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x00001f74] [2018-07-02 21:20:42,724] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:3869 )
  INFO root [0x00001f74] [2018-07-02 21:20:42,725] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:3869, SOMAXCONN=128.
  INFO root [0x00001f74] [2018-07-02 21:20:42,725] - server-configs:
  INFO root [0x00001f74] [2018-07-02 21:20:42,725] - 	gameUpdateHertz : 10
  INFO root [0x00001f74] [2018-07-02 21:20:42,726] - 	entryScriptFile : kbemain
  INFO root [0x00001f74] [2018-07-02 21:20:42,726] - 	internalAddr : 192.168.31.56:3869
  INFO root [0x00001f74] [2018-07-02 21:20:42,726] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x00001f74] [2018-07-02 21:20:42,727] - 	componentID : 7001
  INFO root [0x00001f74] [2018-07-02 21:20:43,828] - Components::findLogger: found logger, addr:192.168.31.56:3868
  INFO root [0x00001f74] [2018-07-02 21:20:43,830] - Components::addComponent: reset orderLog, uid=21684!
  INFO root [0x00001f74] [2018-07-02 21:20:43,831] - Components::addComponent[logger], uid=21684, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001f74] [2018-07-02 21:20:43,846] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 21684. PID: 5928 ----
  INFO root [0x00001f74] [2018-07-02 21:20:43,847] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00001f74] [2018-07-02 21:20:43,848] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001f74] [2018-07-02 21:20:43,849] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001f74] [2018-07-02 21:20:44,006] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001f74] [2018-07-02 21:20:44,006] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001f74] [2018-07-02 21:20:44,007] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001f74] [2018-07-02 21:20:44,012] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001f74] [2018-07-02 21:20:44,015] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001f74] [2018-07-02 21:20:44,015] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001f74] [2018-07-02 21:20:44,016] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001f74] [2018-07-02 21:20:44,098] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001f74] [2018-07-02 21:20:44,206] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001f74] [2018-07-02 21:20:44,207] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001f74] [2018-07-02 21:20:44,579] - loaded script:Account(1).
 DEBUG root [0x00001f74] [2018-07-02 21:20:44,618] - loaded script:Hall(2).
 DEBUG root [0x00001f74] [2018-07-02 21:20:44,658] - loaded script:BattleField(3).
 DEBUG root [0x00001f74] [2018-07-02 21:20:44,705] - loaded script:Avatar(4).
  INFO root [0x00001f74] [2018-07-02 21:20:44,715] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00001f74] [2018-07-02 21:20:44,717] - TelnetServer server is running on port 40000
  INFO root [0x00001f74] [2018-07-02 21:20:46,760] - ---- baseapp is running ----
 DEBUG root [0x00001f74] [2018-07-02 21:20:46,761] - Components::process(): Request for the process of identity...
  WARN root [0x00001f74] [2018-07-02 21:20:52,867] - Forwarding logs to logger[192.168.31.56:3868/0/0/0]...
 ERROR root [0x00001f74] [2018-07-02 21:27:18,592] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00001f74] [2018-07-02 21:27:18,617] - The following logs sent to logger failed:
  WARN root [0x00001f74] [2018-07-02 21:27:18,617] - ==> [2018-07-02 21:27:18 588] TCPPacketReceiver::processPendingEvents(192.168.31.56:3868): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001f74] [2018-07-02 21:27:18,618] - ==> [2018-07-02 21:27:18 592] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00001f74] [2018-07-02 21:27:18,718] - TCPPacketReceiver::processPendingEvents(192.168.31.56:3866): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001f74] [2018-07-02 21:27:18,721] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00001f74] [2018-07-02 21:27:18,734] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00001f74] [2018-07-02 21:27:18,735] - onBaseAppShutDown: state=0
  INFO root [0x00001f74] [2018-07-02 21:27:18,736] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00001f74] [2018-07-02 21:27:18,736] - onBaseAppShutDown: state=1
  INFO root [0x00001f74] [2018-07-02 21:27:18,737] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00001f74] [2018-07-02 21:27:18,737] - onBaseAppShutDown: state=2
 ERROR root [0x00001f74] [2018-07-02 21:27:18,769] - Account::onCellWriteToDBCompleted(2): not found dbmgr!
 ERROR root [0x00001f74] [2018-07-02 21:27:18,771] - Base::onDestroy: not found dbmgr!
 ERROR root [0x00001f74] [2018-07-02 21:27:18,772] - Account::onCellWriteToDBCompleted(3): not found dbmgr!
 ERROR root [0x00001f74] [2018-07-02 21:27:18,774] - Base::onDestroy: not found dbmgr!
 DEBUG root [0x00001f74] [2018-07-02 21:27:18,775] - BattleField::~BattleField(): 4
 DEBUG root [0x00001f74] [2018-07-02 21:27:18,776] - Hall::~Hall(): 1
 ERROR root [0x00001f74] [2018-07-02 21:27:18,776] - EntityGarbages::~EntityGarbages(): leaked, size=4.
 ERROR root [0x00001f74] [2018-07-02 21:27:18,778] - 	--> leaked: Account(2).
 ERROR root [0x00001f74] [2018-07-02 21:27:18,779] - 	--> leaked: Account(3).
 ERROR root [0x00001f74] [2018-07-02 21:27:18,781] - 	--> leaked: Avatar(5).
 ERROR root [0x00001f74] [2018-07-02 21:27:18,781] - 	--> leaked: Avatar(6).
 ERROR root [0x00001f74] [2018-07-02 21:27:18,785] - PyGC::debugTracing(): FixedArray : leaked(14)
 ERROR root [0x00001f74] [2018-07-02 21:27:18,786] - PyGC::debugTracing(): Base : leaked(4)
 ERROR root [0x00001f74] [2018-07-02 21:27:18,787] - PyGC::debugTracing(): FixedDict : leaked(8)
 ERROR root [0x00001f74] [2018-07-02 21:27:18,789] - PyGC::debugTracing(): EntityMailbox : leaked(5)
 DEBUG root [0x00001f74] [2018-07-02 21:27:18,790] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001f74] [2018-07-02 21:27:18,790] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00001f74] [2018-07-02 21:27:18,791] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00001f74] [2018-07-02 21:27:18,791] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001f74] [2018-07-02 21:27:18,791] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00001f74] [2018-07-02 21:27:18,792] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001f74] [2018-07-02 21:27:18,792] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001f74] [2018-07-02 21:27:18,793] - Script::uninstall_py_dlls(): PyInit_unicodedata
 DEBUG root [0x00001f74] [2018-07-02 21:27:18,798] - Avatar::~Avatar(): 6
 DEBUG root [0x00001f74] [2018-07-02 21:27:18,799] - Account::~Account(): 2
 DEBUG root [0x00001f74] [2018-07-02 21:27:18,800] - Avatar::~Avatar(): 5
 DEBUG root [0x00001f74] [2018-07-02 21:27:18,801] - Account::~Account(): 3
  INFO root [0x00001f74] [2018-07-02 21:27:18,816] - Script::uninstall(): is successfully!
 DEBUG root [0x00001f74] [2018-07-02 21:27:18,816] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00001f74] [2018-07-02 21:27:19,117] - ThreadPool::destroy(): waiting for thread(2)[0x53ded8,0x53de70,], try=1
  INFO root [0x00001ea0] [2018-07-02 21:27:26,558] - 
  INFO root [0x00001ea0] [2018-07-02 21:27:26,559] - -----------------------------------------------------------------------------------------


  INFO root [0x00001ea0] [2018-07-02 21:27:26,562] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00001ea0] [2018-07-02 21:27:26,563] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00001ea0] [2018-07-02 21:27:26,563] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00001ea0] [2018-07-02 21:27:26,581] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x00001ea0] [2018-07-02 21:27:26,582] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x00001ea0] [2018-07-02 21:27:26,591] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:4038 )
  INFO root [0x00001ea0] [2018-07-02 21:27:26,591] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:4038, SOMAXCONN=128.
  INFO root [0x00001ea0] [2018-07-02 21:27:26,592] - server-configs:
  INFO root [0x00001ea0] [2018-07-02 21:27:26,592] - 	gameUpdateHertz : 10
  INFO root [0x00001ea0] [2018-07-02 21:27:26,592] - 	entryScriptFile : kbemain
  INFO root [0x00001ea0] [2018-07-02 21:27:26,592] - 	internalAddr : 192.168.31.56:4038
  INFO root [0x00001ea0] [2018-07-02 21:27:26,593] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x00001ea0] [2018-07-02 21:27:26,593] - 	componentID : 7001
  INFO root [0x00001ea0] [2018-07-02 21:27:28,718] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 23004. PID: 8980 ----
  INFO root [0x00001ea0] [2018-07-02 21:27:28,719] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00001ea0] [2018-07-02 21:27:28,720] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001ea0] [2018-07-02 21:27:28,721] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001ea0] [2018-07-02 21:27:28,925] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001ea0] [2018-07-02 21:27:28,926] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001ea0] [2018-07-02 21:27:28,928] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001ea0] [2018-07-02 21:27:28,936] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001ea0] [2018-07-02 21:27:28,937] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001ea0] [2018-07-02 21:27:28,937] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001ea0] [2018-07-02 21:27:28,939] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001ea0] [2018-07-02 21:27:29,110] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001ea0] [2018-07-02 21:27:29,304] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001ea0] [2018-07-02 21:27:29,305] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001ea0] [2018-07-02 21:27:29,446] - loaded script:Account(1).
 DEBUG root [0x00001ea0] [2018-07-02 21:27:29,449] - loaded script:Hall(2).
 DEBUG root [0x00001ea0] [2018-07-02 21:27:29,453] - loaded script:BattleField(3).
 DEBUG root [0x00001ea0] [2018-07-02 21:27:29,457] - loaded script:Avatar(4).
  INFO root [0x00001ea0] [2018-07-02 21:27:29,460] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00001ea0] [2018-07-02 21:27:29,463] - TelnetServer server is running on port 40000
  INFO root [0x00001ea0] [2018-07-02 21:27:31,111] - ---- baseapp is running ----
 DEBUG root [0x00001ea0] [2018-07-02 21:27:31,112] - Components::process(): Request for the process of identity...
  WARN root [0x00001ea0] [2018-07-02 21:27:37,215] - EntityApp::checkTickPeriod: tick took 6104% (6.10 seconds)!
  INFO root [0x00001ea0] [2018-07-02 21:27:38,268] - Components::findComponents: find logger(1)...
  INFO root [0x00001ea0] [2018-07-02 21:27:38,318] - Components::findComponents: found logger, addr:192.168.31.56:4043
  INFO root [0x00001ea0] [2018-07-02 21:27:38,319] - Components::addComponent: reset orderLog, uid=23004!
  INFO root [0x00001ea0] [2018-07-02 21:27:38,319] - Components::addComponent[logger], uid=23004, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001ea0] [2018-07-02 21:27:38,321] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00001ea0] [2018-07-02 21:27:38,421] - Components::findComponents: found dbmgr, addr:192.168.31.56:4041
  INFO root [0x00001ea0] [2018-07-02 21:27:38,422] - Components::addComponent: reset orderLog, uid=23004!
  INFO root [0x00001ea0] [2018-07-02 21:27:38,422] - Components::addComponent[dbmgr], uid=23004, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001ea0] [2018-07-02 21:27:38,423] - Components::findComponents: found dbmgr, addr:192.168.31.56:4041
  WARN root [0x00001ea0] [2018-07-02 21:27:39,423] - Forwarding logs to logger[192.168.31.56:4043/0/0/0]...
 ERROR root [0x00001ea0] [2018-07-02 22:40:17,464] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00001ea0] [2018-07-02 22:40:17,480] - The following logs sent to logger failed:
  WARN root [0x00001ea0] [2018-07-02 22:40:17,481] - ==> [2018-07-02 22:40:17 462] TCPPacketReceiver::processPendingEvents(192.168.31.56:4043): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001ea0] [2018-07-02 22:40:17,482] - ==> [2018-07-02 22:40:17 465] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00001ea0] [2018-07-02 22:40:17,588] - TCPPacketReceiver::processPendingEvents(192.168.31.56:4041): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001ea0] [2018-07-02 22:40:17,593] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00001ea0] [2018-07-02 22:40:17,597] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00001ea0] [2018-07-02 22:40:17,598] - onBaseAppShutDown: state=0
  INFO root [0x00001ea0] [2018-07-02 22:40:17,598] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00001ea0] [2018-07-02 22:40:17,598] - onBaseAppShutDown: state=1
  INFO root [0x00001ea0] [2018-07-02 22:40:17,599] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00001ea0] [2018-07-02 22:40:17,599] - onBaseAppShutDown: state=2
 ERROR root [0x00001ea0] [2018-07-02 22:40:17,635] - Account::onCellWriteToDBCompleted(2002): not found dbmgr!
 ERROR root [0x00001ea0] [2018-07-02 22:40:17,637] - Base::onDestroy: not found dbmgr!
 ERROR root [0x00001ea0] [2018-07-02 22:40:17,639] - Account::onCellWriteToDBCompleted(2003): not found dbmgr!
 ERROR root [0x00001ea0] [2018-07-02 22:40:17,642] - Base::onDestroy: not found dbmgr!
 DEBUG root [0x00001ea0] [2018-07-02 22:40:17,643] - BattleField::~BattleField(): 2004
 DEBUG root [0x00001ea0] [2018-07-02 22:40:17,644] - Hall::~Hall(): 2001
 ERROR root [0x00001ea0] [2018-07-02 22:40:17,644] - EntityGarbages::~EntityGarbages(): leaked, size=4.
 ERROR root [0x00001ea0] [2018-07-02 22:40:17,646] - 	--> leaked: Account(2002).
 ERROR root [0x00001ea0] [2018-07-02 22:40:17,647] - 	--> leaked: Account(2003).
 ERROR root [0x00001ea0] [2018-07-02 22:40:17,650] - 	--> leaked: Avatar(2005).
 ERROR root [0x00001ea0] [2018-07-02 22:40:17,652] - 	--> leaked: Avatar(2006).
 ERROR root [0x00001ea0] [2018-07-02 22:40:17,658] - PyGC::debugTracing(): FixedArray : leaked(14)
 ERROR root [0x00001ea0] [2018-07-02 22:40:17,660] - PyGC::debugTracing(): Base : leaked(4)
 ERROR root [0x00001ea0] [2018-07-02 22:40:17,661] - PyGC::debugTracing(): FixedDict : leaked(8)
 ERROR root [0x00001ea0] [2018-07-02 22:40:17,662] - PyGC::debugTracing(): EntityMailbox : leaked(6)
 DEBUG root [0x00001ea0] [2018-07-02 22:40:17,664] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001ea0] [2018-07-02 22:40:17,664] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00001ea0] [2018-07-02 22:40:17,664] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00001ea0] [2018-07-02 22:40:17,665] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001ea0] [2018-07-02 22:40:17,665] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00001ea0] [2018-07-02 22:40:17,665] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001ea0] [2018-07-02 22:40:17,666] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001ea0] [2018-07-02 22:40:17,666] - Script::uninstall_py_dlls(): PyInit_unicodedata
 DEBUG root [0x00001ea0] [2018-07-02 22:40:17,672] - Avatar::~Avatar(): 2006
 DEBUG root [0x00001ea0] [2018-07-02 22:40:17,673] - Account::~Account(): 2002
 DEBUG root [0x00001ea0] [2018-07-02 22:40:17,673] - Avatar::~Avatar(): 2005
 DEBUG root [0x00001ea0] [2018-07-02 22:40:17,674] - Account::~Account(): 2003
  INFO root [0x00001ea0] [2018-07-02 22:40:17,693] - Script::uninstall(): is successfully!
 DEBUG root [0x00001ea0] [2018-07-02 22:40:17,694] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00000db0] [2018-07-02 22:48:37,648] - 
  INFO root [0x00000db0] [2018-07-02 22:48:37,672] - -----------------------------------------------------------------------------------------


  INFO root [0x00000db0] [2018-07-02 22:48:37,674] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00000db0] [2018-07-02 22:48:37,675] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00000db0] [2018-07-02 22:48:37,676] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00000db0] [2018-07-02 22:48:37,714] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x00000db0] [2018-07-02 22:48:37,715] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x00000db0] [2018-07-02 22:48:37,733] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:4466 )
  INFO root [0x00000db0] [2018-07-02 22:48:37,734] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:4466, SOMAXCONN=128.
  INFO root [0x00000db0] [2018-07-02 22:48:37,734] - server-configs:
  INFO root [0x00000db0] [2018-07-02 22:48:37,734] - 	gameUpdateHertz : 10
  INFO root [0x00000db0] [2018-07-02 22:48:37,735] - 	entryScriptFile : kbemain
  INFO root [0x00000db0] [2018-07-02 22:48:37,735] - 	internalAddr : 192.168.31.56:4466
  INFO root [0x00000db0] [2018-07-02 22:48:37,736] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x00000db0] [2018-07-02 22:48:37,736] - 	componentID : 7001
  INFO root [0x00000db0] [2018-07-02 22:48:39,874] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 6142. PID: 3344 ----
  INFO root [0x00000db0] [2018-07-02 22:48:39,875] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00000db0] [2018-07-02 22:48:39,876] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00000db0] [2018-07-02 22:48:39,877] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00000db0] [2018-07-02 22:48:40,129] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00000db0] [2018-07-02 22:48:40,131] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00000db0] [2018-07-02 22:48:40,132] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00000db0] [2018-07-02 22:48:40,141] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00000db0] [2018-07-02 22:48:40,142] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00000db0] [2018-07-02 22:48:40,143] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00000db0] [2018-07-02 22:48:40,180] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00000db0] [2018-07-02 22:48:40,398] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00000db0] [2018-07-02 22:48:40,561] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00000db0] [2018-07-02 22:48:40,562] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00000db0] [2018-07-02 22:48:40,746] - loaded script:Account(1).
 DEBUG root [0x00000db0] [2018-07-02 22:48:40,750] - loaded script:Hall(2).
 DEBUG root [0x00000db0] [2018-07-02 22:48:40,755] - loaded script:BattleField(3).
 DEBUG root [0x00000db0] [2018-07-02 22:48:40,759] - loaded script:Avatar(4).
  INFO root [0x00000db0] [2018-07-02 22:48:40,762] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00000db0] [2018-07-02 22:48:40,769] - TelnetServer server is running on port 40000
  INFO root [0x00000db0] [2018-07-02 22:48:41,866] - ---- baseapp is running ----
 DEBUG root [0x00000db0] [2018-07-02 22:48:41,867] - Components::process(): Request for the process of identity...
  WARN root [0x00000db0] [2018-07-02 22:48:47,969] - EntityApp::checkTickPeriod: tick took 6101% (6.10 seconds)!
  INFO root [0x00000db0] [2018-07-02 22:48:48,992] - Components::findComponents: find logger(1)...
  INFO root [0x00000db0] [2018-07-02 22:48:48,997] - Components::findComponents: found logger, addr:192.168.31.56:4465
  INFO root [0x00000db0] [2018-07-02 22:48:48,998] - Components::addComponent: reset orderLog, uid=6142!
  INFO root [0x00000db0] [2018-07-02 22:48:48,998] - Components::addComponent[logger], uid=6142, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00000db0] [2018-07-02 22:48:49,000] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00000db0] [2018-07-02 22:48:49,102] - Components::findComponents: found dbmgr, addr:192.168.31.56:4467
  INFO root [0x00000db0] [2018-07-02 22:48:49,102] - Components::addComponent: reset orderLog, uid=6142!
  INFO root [0x00000db0] [2018-07-02 22:48:49,103] - Components::addComponent[dbmgr], uid=6142, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00000db0] [2018-07-02 22:48:49,104] - Components::findComponents: found dbmgr, addr:192.168.31.56:4467
  WARN root [0x00000db0] [2018-07-02 22:48:50,105] - Forwarding logs to logger[192.168.31.56:4465/0/0/0]...
  INFO root [0x0000234c] [2018-07-02 23:08:05,266] - 
  INFO root [0x0000234c] [2018-07-02 23:08:05,273] - -----------------------------------------------------------------------------------------


  INFO root [0x0000234c] [2018-07-02 23:08:05,278] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x0000234c] [2018-07-02 23:08:05,279] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x0000234c] [2018-07-02 23:08:05,280] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x0000234c] [2018-07-02 23:08:05,304] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20016 )
  INFO root [0x0000234c] [2018-07-02 23:08:05,305] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20016, SOMAXCONN=128.
  INFO root [0x0000234c] [2018-07-02 23:08:05,310] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:5165 )
  INFO root [0x0000234c] [2018-07-02 23:08:05,311] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:5165, SOMAXCONN=128.
  INFO root [0x0000234c] [2018-07-02 23:08:05,311] - server-configs:
  INFO root [0x0000234c] [2018-07-02 23:08:05,311] - 	gameUpdateHertz : 10
  INFO root [0x0000234c] [2018-07-02 23:08:05,312] - 	entryScriptFile : kbemain
  INFO root [0x0000234c] [2018-07-02 23:08:05,312] - 	internalAddr : 192.168.31.56:5165
  INFO root [0x0000234c] [2018-07-02 23:08:05,312] - 	externalAddr : 192.168.31.56:20016
  INFO root [0x0000234c] [2018-07-02 23:08:05,314] - 	componentID : 7001
  INFO root [0x0000234c] [2018-07-02 23:08:07,455] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 9956. PID: 3564 ----
  INFO root [0x0000234c] [2018-07-02 23:08:07,459] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x0000234c] [2018-07-02 23:08:07,467] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x0000234c] [2018-07-02 23:08:07,468] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x0000234c] [2018-07-02 23:08:07,743] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x0000234c] [2018-07-02 23:08:07,744] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x0000234c] [2018-07-02 23:08:07,746] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x0000234c] [2018-07-02 23:08:07,754] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x0000234c] [2018-07-02 23:08:07,755] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x0000234c] [2018-07-02 23:08:07,755] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x0000234c] [2018-07-02 23:08:07,757] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x0000234c] [2018-07-02 23:08:07,891] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x0000234c] [2018-07-02 23:08:07,981] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x0000234c] [2018-07-02 23:08:07,981] - Script::install(): KBExtra is successfully!
 DEBUG root [0x0000234c] [2018-07-02 23:08:08,120] - loaded script:Account(1).
 DEBUG root [0x0000234c] [2018-07-02 23:08:08,124] - loaded script:Hall(2).
 DEBUG root [0x0000234c] [2018-07-02 23:08:08,128] - loaded script:BattleField(3).
 DEBUG root [0x0000234c] [2018-07-02 23:08:08,131] - loaded script:Avatar(4).
  INFO root [0x0000234c] [2018-07-02 23:08:08,135] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x0000234c] [2018-07-02 23:08:08,137] - TelnetServer server is running on port 40001
  INFO root [0x0000234c] [2018-07-02 23:08:09,192] - ---- baseapp is running ----
 DEBUG root [0x0000234c] [2018-07-02 23:08:09,193] - Components::process(): Request for the process of identity...
 ERROR root [0x0000234c] [2018-07-02 23:08:09,347] - Components::process: found baseapp, addr:192.168.31.56:4466
 ERROR root [0x0000234c] [2018-07-02 23:08:09,350] - ServerApp::onIdentityillegal: The current process and baseapp(componentID=7001 ->conflicted???, pid=3344, addr=192.168.31.56) conflict, the process will exit!
Can modify the components-CID and UID to avoid conflict.
  INFO root [0x0000234c] [2018-07-02 23:08:09,352] - Shutdowner::onShutdownBegin: shutting down(period=1, tickPeriod=1)
S_INFO root [0x0000234c] [2018-07-02 23:08:09,353] - onBaseAppShutDown: state=0
  INFO root [0x0000234c] [2018-07-02 23:08:10,370] - Shutdowner::onShutdown: shutting down
S_INFO root [0x0000234c] [2018-07-02 23:08:10,371] - onBaseAppShutDown: state=1
S_INFO root [0x0000234c] [2018-07-02 23:08:11,372] - onReadyForShutDown()
  INFO root [0x0000234c] [2018-07-02 23:08:11,373] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x0000234c] [2018-07-02 23:08:11,373] - onBaseAppShutDown: state=2
 DEBUG root [0x0000234c] [2018-07-02 23:08:11,386] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x0000234c] [2018-07-02 23:08:11,386] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x0000234c] [2018-07-02 23:08:11,386] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x0000234c] [2018-07-02 23:08:11,386] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x0000234c] [2018-07-02 23:08:11,387] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x0000234c] [2018-07-02 23:08:11,387] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x0000234c] [2018-07-02 23:08:11,387] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x0000234c] [2018-07-02 23:08:11,387] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x0000234c] [2018-07-02 23:08:11,400] - Script::uninstall(): is successfully!
 DEBUG root [0x0000234c] [2018-07-02 23:08:11,400] - ThreadPool::destroy(): starting size 2.
  WARN root [0x0000234c] [2018-07-02 23:08:11,701] - ThreadPool::destroy(): waiting for thread(2)[0x1b7ded0,0x1b7de68,], try=1
 DEBUG root [0x0000234c] [2018-07-02 23:08:12,101] - TPThread::~TPThread(): 0x1b7ded0
 DEBUG root [0x0000234c] [2018-07-02 23:08:12,101] - TPThread::~TPThread(): 0x1b7de68
 DEBUG root [0x0000234c] [2018-07-02 23:08:12,101] - ThreadPool::destroy(): successfully!
 DEBUG root [0x0000234c] [2018-07-02 23:08:12,102] - Bundle::destroyObjPool(): size 5.
 DEBUG root [0x0000234c] [2018-07-02 23:08:12,102] - Channel::destroyObjPool(): size 0.
 DEBUG root [0x0000234c] [2018-07-02 23:08:12,102] - TCPPacket::destroyObjPool(): size 0.
 DEBUG root [0x0000234c] [2018-07-02 23:08:12,102] - UDPPacket::destroyObjPool(): size 12.
 DEBUG root [0x0000234c] [2018-07-02 23:08:12,103] - EndPoint::destroyObjPool(): size 0.
 DEBUG root [0x0000234c] [2018-07-02 23:08:12,103] - Address::destroyObjPool(): size 0.
 DEBUG root [0x0000234c] [2018-07-02 23:08:12,103] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [0x0000234c] [2018-07-02 23:08:12,103] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [0x0000234c] [2018-07-02 23:08:12,103] - baseapp(7001) has shut down.
  INFO root [0x0000234c] [2018-07-02 23:08:13,127] - EventDispatcher()::~EventDispatcher: Num timers = 2
  INFO root [0x00000d78] [2018-07-02 23:10:24,835] - 
  INFO root [0x00000d78] [2018-07-02 23:10:24,840] - -----------------------------------------------------------------------------------------


  INFO root [0x00000d78] [2018-07-02 23:10:24,843] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00000d78] [2018-07-02 23:10:24,843] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00000d78] [2018-07-02 23:10:24,844] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00000d78] [2018-07-02 23:10:24,873] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20016 )
  INFO root [0x00000d78] [2018-07-02 23:10:24,874] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20016, SOMAXCONN=128.
  INFO root [0x00000d78] [2018-07-02 23:10:24,897] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:5196 )
  INFO root [0x00000d78] [2018-07-02 23:10:24,898] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:5196, SOMAXCONN=128.
  INFO root [0x00000d78] [2018-07-02 23:10:24,898] - server-configs:
  INFO root [0x00000d78] [2018-07-02 23:10:24,898] - 	gameUpdateHertz : 10
  INFO root [0x00000d78] [2018-07-02 23:10:24,899] - 	entryScriptFile : kbemain
  INFO root [0x00000d78] [2018-07-02 23:10:24,899] - 	internalAddr : 192.168.31.56:5196
  INFO root [0x00000d78] [2018-07-02 23:10:24,899] - 	externalAddr : 192.168.31.56:20016
  INFO root [0x00000d78] [2018-07-02 23:10:24,899] - 	componentID : 7001
  INFO root [0x00000d78] [2018-07-02 23:10:25,836] - Components::findLogger: found logger, addr:192.168.31.56:5198
  INFO root [0x00000d78] [2018-07-02 23:10:25,837] - Components::addComponent: reset orderLog, uid=10410!
  INFO root [0x00000d78] [2018-07-02 23:10:25,837] - Components::addComponent[logger], uid=10410, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00000d78] [2018-07-02 23:10:25,852] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 10410. PID: 4136 ----
  INFO root [0x00000d78] [2018-07-02 23:10:25,853] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00000d78] [2018-07-02 23:10:25,854] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00000d78] [2018-07-02 23:10:25,855] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00000d78] [2018-07-02 23:10:26,021] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00000d78] [2018-07-02 23:10:26,023] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00000d78] [2018-07-02 23:10:26,024] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00000d78] [2018-07-02 23:10:26,032] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00000d78] [2018-07-02 23:10:26,033] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00000d78] [2018-07-02 23:10:26,034] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00000d78] [2018-07-02 23:10:26,035] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00000d78] [2018-07-02 23:10:26,112] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00000d78] [2018-07-02 23:10:26,195] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00000d78] [2018-07-02 23:10:26,195] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00000d78] [2018-07-02 23:10:26,303] - loaded script:Account(1).
 DEBUG root [0x00000d78] [2018-07-02 23:10:26,306] - loaded script:Hall(2).
 DEBUG root [0x00000d78] [2018-07-02 23:10:26,309] - loaded script:BattleField(3).
 DEBUG root [0x00000d78] [2018-07-02 23:10:26,313] - loaded script:Avatar(4).
  INFO root [0x00000d78] [2018-07-02 23:10:26,316] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00000d78] [2018-07-02 23:10:26,318] - TelnetServer server is running on port 40001
  INFO root [0x00000d78] [2018-07-02 23:10:28,042] - ---- baseapp is running ----
 DEBUG root [0x00000d78] [2018-07-02 23:10:28,045] - Components::process(): Request for the process of identity...
  WARN root [0x00000d78] [2018-07-02 23:10:28,358] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20388, errorno=10048
 ERROR root [0x00000d78] [2018-07-02 23:10:28,590] - Components::process: found baseapp, addr:192.168.31.56:4466
 ERROR root [0x00000d78] [2018-07-02 23:10:28,593] - ServerApp::onIdentityillegal: The current process and baseapp(componentID=7001 ->conflicted???, pid=3344, addr=192.168.31.56) conflict, the process will exit!
Can modify the components-CID and UID to avoid conflict.
  INFO root [0x00000d78] [2018-07-02 23:10:28,595] - Shutdowner::onShutdownBegin: shutting down(period=1, tickPeriod=1)
S_INFO root [0x00000d78] [2018-07-02 23:10:28,596] - onBaseAppShutDown: state=0
  WARN root [0x00000d78] [2018-07-02 23:10:28,597] - Forwarding logs to logger[192.168.31.56:5198/0/0/0]...
  INFO root [0x00000d78] [2018-07-02 23:10:32,335] - EventDispatcher()::~EventDispatcher: Num timers = 1
 ERROR root [0x00000db0] [2018-07-02 23:12:39,776] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=6
  INFO root [0x00000db0] [2018-07-02 23:12:39,777] - The following logs sent to logger failed:
  WARN root [0x00000db0] [2018-07-02 23:12:39,778] - ==> [2018-07-02 23:12:39 776] TCPPacketReceiver::processPendingEvents(192.168.31.56:4465): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00000db0] [2018-07-02 23:12:39,778] - ==> [2018-07-02 23:12:39 777] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=6
  WARN root [0x00000db0] [2018-07-02 23:12:43,531] - TCPPacketReceiver::processPendingEvents(192.168.31.56:4463): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00000db0] [2018-07-02 23:12:43,532] - Components::removeComponentByChannel: baseappmgr : 5000, Abnormal exit! errorno=6
  INFO root [0x00000db0] [2018-07-02 23:12:43,532] - Shutdowner::onShutdownBegin: shutting down(period=1, tickPeriod=1)
S_INFO root [0x00000db0] [2018-07-02 23:12:43,532] - onBaseAppShutDown: state=0
  INFO root [0x00000db0] [2018-07-02 23:12:44,533] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00000db0] [2018-07-02 23:12:44,533] - onBaseAppShutDown: state=1
 DEBUG root [0x00000db0] [2018-07-02 23:12:44,534] - Account::~Account(): 3
 DEBUG root [0x00000db0] [2018-07-02 23:12:44,534] - Account::~Account(): 2
 DEBUG root [0x00000db0] [2018-07-02 23:12:44,534] - BattleField::~BattleField(): 4
S_INFO root [0x00000db0] [2018-07-02 23:12:45,535] - onReadyForShutDown()
  INFO root [0x00000db0] [2018-07-02 23:12:45,535] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00000db0] [2018-07-02 23:12:45,536] - onBaseAppShutDown: state=2
 DEBUG root [0x00000db0] [2018-07-02 23:12:45,548] - Hall::~Hall(): 1
 DEBUG root [0x00000db0] [2018-07-02 23:12:45,549] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00000db0] [2018-07-02 23:12:45,549] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00000db0] [2018-07-02 23:12:45,549] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00000db0] [2018-07-02 23:12:45,550] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00000db0] [2018-07-02 23:12:45,550] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00000db0] [2018-07-02 23:12:45,550] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00000db0] [2018-07-02 23:12:45,550] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00000db0] [2018-07-02 23:12:45,550] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00000db0] [2018-07-02 23:12:45,562] - Script::uninstall(): is successfully!
 DEBUG root [0x00000db0] [2018-07-02 23:12:45,563] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00000db0] [2018-07-02 23:12:45,863] - ThreadPool::destroy(): waiting for thread(2)[0x1bf6720,0x1bf66b8,], try=1
 DEBUG root [0x00000db0] [2018-07-02 23:12:46,264] - TPThread::~TPThread(): 0x1bf6720
 DEBUG root [0x00000db0] [2018-07-02 23:12:46,264] - TPThread::~TPThread(): 0x1bf66b8
 DEBUG root [0x00000db0] [2018-07-02 23:12:46,264] - ThreadPool::destroy(): successfully!
 DEBUG root [0x00000db0] [2018-07-02 23:12:46,265] - Bundle::destroyObjPool(): size 5.
 DEBUG root [0x00000db0] [2018-07-02 23:12:46,265] - Channel::destroyObjPool(): size 14.
 DEBUG root [0x00000db0] [2018-07-02 23:12:46,265] - TCPPacket::destroyObjPool(): size 0.
 DEBUG root [0x00000db0] [2018-07-02 23:12:46,266] - UDPPacket::destroyObjPool(): size 14.
 DEBUG root [0x00000db0] [2018-07-02 23:12:46,266] - EndPoint::destroyObjPool(): size 14.
 DEBUG root [0x00000db0] [2018-07-02 23:12:46,266] - Address::destroyObjPool(): size 0.
 DEBUG root [0x00000db0] [2018-07-02 23:12:46,266] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [0x00000db0] [2018-07-02 23:12:46,266] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [0x00000db0] [2018-07-02 23:12:46,266] - baseapp(7001) has shut down.
  INFO root [0x00000db0] [2018-07-02 23:12:47,268] - EventDispatcher()::~EventDispatcher: Num timers = 1
  INFO root [0x000012c4] [2018-07-02 23:13:40,101] - 
  INFO root [0x000012c4] [2018-07-02 23:13:40,117] - -----------------------------------------------------------------------------------------


  INFO root [0x000012c4] [2018-07-02 23:13:40,119] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x000012c4] [2018-07-02 23:13:40,120] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x000012c4] [2018-07-02 23:13:40,120] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x000012c4] [2018-07-02 23:13:40,184] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x000012c4] [2018-07-02 23:13:40,184] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x000012c4] [2018-07-02 23:13:40,189] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:5240 )
  INFO root [0x000012c4] [2018-07-02 23:13:40,189] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:5240, SOMAXCONN=128.
  INFO root [0x000012c4] [2018-07-02 23:13:40,189] - server-configs:
  INFO root [0x000012c4] [2018-07-02 23:13:40,190] - 	gameUpdateHertz : 10
  INFO root [0x000012c4] [2018-07-02 23:13:40,190] - 	entryScriptFile : kbemain
  INFO root [0x000012c4] [2018-07-02 23:13:40,190] - 	internalAddr : 192.168.31.56:5240
  INFO root [0x000012c4] [2018-07-02 23:13:40,191] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x000012c4] [2018-07-02 23:13:40,191] - 	componentID : 7001
  INFO root [0x000012c4] [2018-07-02 23:13:41,025] - Components::findLogger: found logger, addr:192.168.31.56:5243
  INFO root [0x000012c4] [2018-07-02 23:13:41,025] - Components::addComponent: reset orderLog, uid=11050!
  INFO root [0x000012c4] [2018-07-02 23:13:41,026] - Components::addComponent[logger], uid=11050, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000012c4] [2018-07-02 23:13:41,039] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 11050. PID: 9784 ----
  INFO root [0x000012c4] [2018-07-02 23:13:41,040] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x000012c4] [2018-07-02 23:13:41,041] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x000012c4] [2018-07-02 23:13:41,041] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x000012c4] [2018-07-02 23:13:41,263] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000012c4] [2018-07-02 23:13:41,265] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x000012c4] [2018-07-02 23:13:41,266] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x000012c4] [2018-07-02 23:13:41,285] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x000012c4] [2018-07-02 23:13:41,286] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x000012c4] [2018-07-02 23:13:41,287] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x000012c4] [2018-07-02 23:13:41,288] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x000012c4] [2018-07-02 23:13:41,382] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x000012c4] [2018-07-02 23:13:41,470] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x000012c4] [2018-07-02 23:13:41,471] - Script::install(): KBExtra is successfully!
 DEBUG root [0x000012c4] [2018-07-02 23:13:41,595] - loaded script:Account(1).
 DEBUG root [0x000012c4] [2018-07-02 23:13:41,599] - loaded script:Hall(2).
 DEBUG root [0x000012c4] [2018-07-02 23:13:41,602] - loaded script:BattleField(3).
 DEBUG root [0x000012c4] [2018-07-02 23:13:41,606] - loaded script:Avatar(4).
  INFO root [0x000012c4] [2018-07-02 23:13:41,609] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x000012c4] [2018-07-02 23:13:41,611] - TelnetServer server is running on port 40000
  INFO root [0x000012c4] [2018-07-02 23:13:43,227] - ---- baseapp is running ----
 DEBUG root [0x000012c4] [2018-07-02 23:13:43,231] - Components::process(): Request for the process of identity...
  WARN root [0x000012c4] [2018-07-02 23:13:49,364] - Forwarding logs to logger[192.168.31.56:5243/0/0/0]...
 ERROR root [0x000012c4] [2018-07-02 23:15:43,612] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x000012c4] [2018-07-02 23:15:43,647] - The following logs sent to logger failed:
  WARN root [0x000012c4] [2018-07-02 23:15:43,648] - ==> [2018-07-02 23:15:43 606] TCPPacketReceiver::processPendingEvents(192.168.31.56:5243): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000012c4] [2018-07-02 23:15:43,648] - ==> [2018-07-02 23:15:43 614] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x000012c4] [2018-07-02 23:15:43,722] - TCPPacketReceiver::processPendingEvents(192.168.31.56:5236): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000012c4] [2018-07-02 23:15:43,725] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x000012c4] [2018-07-02 23:15:43,726] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x000012c4] [2018-07-02 23:15:43,727] - onBaseAppShutDown: state=0
  INFO root [0x000012c4] [2018-07-02 23:15:43,728] - Shutdowner::onShutdown: shutting down
S_INFO root [0x000012c4] [2018-07-02 23:15:43,729] - onBaseAppShutDown: state=1
  INFO root [0x000012c4] [2018-07-02 23:15:43,729] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x000012c4] [2018-07-02 23:15:43,730] - onBaseAppShutDown: state=2
 DEBUG root [0x000012c4] [2018-07-02 23:15:43,763] - BattleField::~BattleField(): 4
 DEBUG root [0x000012c4] [2018-07-02 23:15:43,764] - Hall::~Hall(): 1
 ERROR root [0x000012c4] [2018-07-02 23:15:43,765] - EntityGarbages::~EntityGarbages(): leaked, size=4.
 ERROR root [0x000012c4] [2018-07-02 23:15:43,779] - 	--> leaked: Account(3).
 ERROR root [0x000012c4] [2018-07-02 23:15:43,781] - 	--> leaked: Account(2).
 ERROR root [0x000012c4] [2018-07-02 23:15:43,783] - 	--> leaked: Avatar(5).
 ERROR root [0x000012c4] [2018-07-02 23:15:43,785] - 	--> leaked: Avatar(6).
 ERROR root [0x000012c4] [2018-07-02 23:15:43,789] - PyGC::debugTracing(): FixedArray : leaked(14)
 ERROR root [0x000012c4] [2018-07-02 23:15:43,791] - PyGC::debugTracing(): Base : leaked(4)
 ERROR root [0x000012c4] [2018-07-02 23:15:43,793] - PyGC::debugTracing(): FixedDict : leaked(8)
 ERROR root [0x000012c4] [2018-07-02 23:15:43,796] - PyGC::debugTracing(): EntityMailbox : leaked(5)
 DEBUG root [0x000012c4] [2018-07-02 23:15:43,798] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000012c4] [2018-07-02 23:15:43,798] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x000012c4] [2018-07-02 23:15:43,799] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x000012c4] [2018-07-02 23:15:43,799] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x000012c4] [2018-07-02 23:15:43,800] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x000012c4] [2018-07-02 23:15:43,800] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x000012c4] [2018-07-02 23:15:43,800] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x000012c4] [2018-07-02 23:15:43,801] - Script::uninstall_py_dlls(): PyInit_unicodedata
 DEBUG root [0x000012c4] [2018-07-02 23:15:43,806] - Avatar::~Avatar(): 6
 DEBUG root [0x000012c4] [2018-07-02 23:15:43,807] - Account::~Account(): 2
 DEBUG root [0x000012c4] [2018-07-02 23:15:43,808] - Avatar::~Avatar(): 5
 DEBUG root [0x000012c4] [2018-07-02 23:15:43,809] - Account::~Account(): 3
  INFO root [0x000012c4] [2018-07-02 23:15:43,826] - Script::uninstall(): is successfully!
 DEBUG root [0x000012c4] [2018-07-02 23:15:43,827] - ThreadPool::destroy(): starting size 2.
